<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod46</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                        </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod46</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>59 genes                  </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=29">29</a></p>
</td>
<td class=cellinside>ABR
</td>
<td class=cellinside>ABR, RhoGEF and GTPase activating protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11214">11214</a></p>
</td>
<td class=cellinside>AKAP13
</td>
<td class=cellinside>A-kinase anchoring protein 13
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=116985">116985</a></p>
</td>
<td class=cellinside>ARAP1
</td>
<td class=cellinside>ArfGAP with RhoGAP domain, ankyrin repeat and PH domain 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64411">64411</a></p>
</td>
<td class=cellinside>ARAP3
</td>
<td class=cellinside>ArfGAP with RhoGAP domain, ankyrin repeat and PH domain 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79658">79658</a></p>
</td>
<td class=cellinside>ARHGAP10
</td>
<td class=cellinside>Rho GTPase activating protein 10
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=94134">94134</a></p>
</td>
<td class=cellinside>ARHGAP12
</td>
<td class=cellinside>Rho GTPase activating protein 12
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55843">55843</a></p>
</td>
<td class=cellinside>ARHGAP15
</td>
<td class=cellinside>Rho GTPase activating protein 15
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=93663">93663</a></p>
</td>
<td class=cellinside>ARHGAP18
</td>
<td class=cellinside>Rho GTPase activating protein 18
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84986">84986</a></p>
</td>
<td class=cellinside>ARHGAP19
</td>
<td class=cellinside>Rho GTPase activating protein 19
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57569">57569</a></p>
</td>
<td class=cellinside>ARHGAP20
</td>
<td class=cellinside>Rho GTPase activating protein 20
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=58504">58504</a></p>
</td>
<td class=cellinside>ARHGAP22
</td>
<td class=cellinside>Rho GTPase activating protein 22
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57636">57636</a></p>
</td>
<td class=cellinside>ARHGAP23
</td>
<td class=cellinside>Rho GTPase activating protein 23
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=83478">83478</a></p>
</td>
<td class=cellinside>ARHGAP24
</td>
<td class=cellinside>Rho GTPase activating protein 24
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9938">9938</a></p>
</td>
<td class=cellinside>ARHGAP25
</td>
<td class=cellinside>Rho GTPase activating protein 25
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79822">79822</a></p>
</td>
<td class=cellinside>ARHGAP28
</td>
<td class=cellinside>Rho GTPase activating protein 28
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9411">9411</a></p>
</td>
<td class=cellinside>ARHGAP29
</td>
<td class=cellinside>Rho GTPase activating protein 29
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=257106">257106</a></p>
</td>
<td class=cellinside>ARHGAP30
</td>
<td class=cellinside>Rho GTPase activating protein 30
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57514">57514</a></p>
</td>
<td class=cellinside>ARHGAP31
</td>
<td class=cellinside>Rho GTPase activating protein 31
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=115703">115703</a></p>
</td>
<td class=cellinside>ARHGAP33
</td>
<td class=cellinside>Rho GTPase activating protein 33
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=158763">158763</a></p>
</td>
<td class=cellinside>ARHGAP36
</td>
<td class=cellinside>Rho GTPase activating protein 36
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80728">80728</a></p>
</td>
<td class=cellinside>ARHGAP39
</td>
<td class=cellinside>Rho GTPase activating protein 39
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=343578">343578</a></p>
</td>
<td class=cellinside>ARHGAP40
</td>
<td class=cellinside>Rho GTPase activating protein 40
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9912">9912</a></p>
</td>
<td class=cellinside>ARHGAP44
</td>
<td class=cellinside>Rho GTPase activating protein 44
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23779">23779</a></p>
</td>
<td class=cellinside>ARHGAP8
</td>
<td class=cellinside>Rho GTPase activating protein 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64333">64333</a></p>
</td>
<td class=cellinside>ARHGAP9
</td>
<td class=cellinside>Rho GTPase activating protein 9
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=396">396</a></p>
</td>
<td class=cellinside>ARHGDIA
</td>
<td class=cellinside>Rho GDP dissociation inhibitor alpha
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=397">397</a></p>
</td>
<td class=cellinside>ARHGDIB
</td>
<td class=cellinside>Rho GDP dissociation inhibitor beta
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=398">398</a></p>
</td>
<td class=cellinside>ARHGDIG
</td>
<td class=cellinside>Rho GDP dissociation inhibitor gamma
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=27237">27237</a></p>
</td>
<td class=cellinside>ARHGEF16
</td>
<td class=cellinside>Rho guanine nucleotide exchange factor 16
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9828">9828</a></p>
</td>
<td class=cellinside>ARHGEF17
</td>
<td class=cellinside>Rho guanine nucleotide exchange factor 17
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23370">23370</a></p>
</td>
<td class=cellinside>ARHGEF18
</td>
<td class=cellinside>Rho/Rac guanine nucleotide exchange factor 18
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9181">9181</a></p>
</td>
<td class=cellinside>ARHGEF2
</td>
<td class=cellinside>Rho/Rac guanine nucleotide exchange factor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=50649">50649</a></p>
</td>
<td class=cellinside>ARHGEF4
</td>
<td class=cellinside>Rho guanine nucleotide exchange factor 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55789">55789</a></p>
</td>
<td class=cellinside>DEPDC1B
</td>
<td class=cellinside>DEP domain containing 1B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=91614">91614</a></p>
</td>
<td class=cellinside>DEPDC7
</td>
<td class=cellinside>DEP domain containing 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10144">10144</a></p>
</td>
<td class=cellinside>FAM13A
</td>
<td class=cellinside>family with sequence similarity 13 member A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51306">51306</a></p>
</td>
<td class=cellinside>FAM13B
</td>
<td class=cellinside>family with sequence similarity 13 member B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2245">2245</a></p>
</td>
<td class=cellinside>FGD1
</td>
<td class=cellinside>FYVE, RhoGEF and PH domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=221472">221472</a></p>
</td>
<td class=cellinside>FGD2
</td>
<td class=cellinside>FYVE, RhoGEF and PH domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=89846">89846</a></p>
</td>
<td class=cellinside>FGD3
</td>
<td class=cellinside>FYVE, RhoGEF and PH domain containing 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=121512">121512</a></p>
</td>
<td class=cellinside>FGD4
</td>
<td class=cellinside>FYVE, RhoGEF and PH domain containing 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51291">51291</a></p>
</td>
<td class=cellinside>GMIP
</td>
<td class=cellinside>GEM interacting protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23526">23526</a></p>
</td>
<td class=cellinside>HMHA1
</td>
<td class=cellinside>Rho GTPase activating protein 45
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8997">8997</a></p>
</td>
<td class=cellinside>KALRN
</td>
<td class=cellinside>kalirin RhoGEF kinase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4168">4168</a></p>
</td>
<td class=cellinside>MCF2
</td>
<td class=cellinside>MCF.2 cell line derived transforming sequence
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23263">23263</a></p>
</td>
<td class=cellinside>MCF2L
</td>
<td class=cellinside>MCF.2 cell line derived transforming sequence like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4983">4983</a></p>
</td>
<td class=cellinside>OPHN1
</td>
<td class=cellinside>oligophrenin 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64857">64857</a></p>
</td>
<td class=cellinside>PLEKHG2
</td>
<td class=cellinside>pleckstrin homology and RhoGEF domain containing G2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57449">57449</a></p>
</td>
<td class=cellinside>PLEKHG5
</td>
<td class=cellinside>pleckstrin homology and RhoGEF domain containing G5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=89941">89941</a></p>
</td>
<td class=cellinside>RHOT2
</td>
<td class=cellinside>ras homolog family member T2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57522">57522</a></p>
</td>
<td class=cellinside>SRGAP1
</td>
<td class=cellinside>SLIT-ROBO Rho GTPase activating protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9901">9901</a></p>
</td>
<td class=cellinside>SRGAP3
</td>
<td class=cellinside>SLIT-ROBO Rho GTPase activating protein 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=90627">90627</a></p>
</td>
<td class=cellinside>STARD13
</td>
<td class=cellinside>StAR related lipid transfer domain containing 13
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9754">9754</a></p>
</td>
<td class=cellinside>STARD8
</td>
<td class=cellinside>StAR related lipid transfer domain containing 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=85360">85360</a></p>
</td>
<td class=cellinside>SYDE1
</td>
<td class=cellinside>synapse defective Rho GTPase homolog 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84144">84144</a></p>
</td>
<td class=cellinside>SYDE2
</td>
<td class=cellinside>synapse defective Rho GTPase homolog 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=117289">117289</a></p>
</td>
<td class=cellinside>TAGAP
</td>
<td class=cellinside>T cell activation RhoGTPase activating protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26230">26230</a></p>
</td>
<td class=cellinside>TIAM2
</td>
<td class=cellinside>T cell lymphoma invasion and metastasis 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9322">9322</a></p>
</td>
<td class=cellinside>TRIP10
</td>
<td class=cellinside>thyroid hormone receptor interactor 10
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>2.15e-01                                                                                                                                              </td>
<td class=cellinside>6.08e-130                                                                                                                                             </td>
<td class=cellinside>Reactome                                                                                                                                              </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-194840&species=Homo+sapiens&species=Entries+without+species&cluster=true">Rho GTPase cycle</a></p></td>
<td class=cellinside>Signal Transduction                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>1.70e-01                                                                                                                                              </td>
<td class=cellinside>7.37e-126                                                                                                                                             </td>
<td class=cellinside>GO                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0051056">regulation of small GTPase mediated signal transduction</a></p>            </td>
<td class=cellinside>biological_process                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>8.44e-03                                                                                                                                              </td>
<td class=cellinside> 3.33e-09                                                                                                                                             </td>
<td class=cellinside>GO                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046847">filopodium assembly</a></p>                                                </td>
<td class=cellinside>biological_process                                                                                                                                    </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod46_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0051056">regulation of small GTPase mediated signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043087">regulation of GTPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043547">positive regulation of GTPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035023">regulation of Rho protein signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046578">regulation of Ras protein signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005096">GTPase activator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008047">enzyme activator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030695">GTPase regulator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.49e-20
</td>
<td class=cellinside>1.80e-17
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005089">Rho guanyl-nucleotide exchange factor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.51e-15
</td>
<td class=cellinside>4.24e-12
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043065">positive regulation of apoptotic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.90e-15
</td>
<td class=cellinside>4.61e-12
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043068">positive regulation of programmed cell death</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.29e-14
</td>
<td class=cellinside>1.21e-11
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0010942">positive regulation of cell death</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.77e-13
</td>
<td class=cellinside>2.62e-10
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005088">Ras guanyl-nucleotide exchange factor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.07e-12
</td>
<td class=cellinside>7.40e-10
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005085">guanyl-nucleotide exchange factor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.58e-11
</td>
<td class=cellinside>3.97e-08
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031252">cell leading edge</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.65e-10
</td>
<td class=cellinside>3.60e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008360">regulation of cell shape</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.97e-10
</td>
<td class=cellinside>3.83e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0022604">regulation of cell morphogenesis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.22e-09
</td>
<td class=cellinside>9.57e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030027">lamellipodium</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.34e-09
</td>
<td class=cellinside>2.32e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046847">filopodium assembly</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.60e-08
</td>
<td class=cellinside>7.08e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031267">small GTPase binding</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>SIGNALING BY RHO GTPASES
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>P75 NTR RECEPTOR MEDIATED SIGNALLING
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>NRAGE SIGNALS DEATH THROUGH JNK
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>SIGNALLING BY NGF
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>G ALPHA1213 SIGNALLING EVENTS
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>CELL DEATH SIGNALLING VIA NRAGE NRIF AND NADE
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>SIGNALING BY RHO GTPASES
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>P75 NTR RECEPTOR MEDIATED SIGNALLING
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>NRAGE SIGNALS DEATH THROUGH JNK
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>SIGNALLING BY NGF
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>G ALPHA1213 SIGNALLING EVENTS
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>CELL DEATH SIGNALLING VIA NRAGE NRIF AND NADE
</td></tr>
 
<tr> 
<td class=cellinside>1.04e-03
</td>
<td class=cellinside>1.04e-01
</td>
<td class=cellinside>SIGNALING BY ROBO RECEPTOR
</td></tr>
 
<tr> 
<td class=cellinside>5.42e-03
</td>
<td class=cellinside>4.88e-01
</td>
<td class=cellinside>SIGNALING BY ROBO RECEPTOR
</td></tr>
 
<tr> 
<td class=cellinside>4.14e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TGF BETA RECEPTOR SIGNALING IN EMT EPITHELIAL TO MESENCHYMAL TRANSITION
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.26e-05
</td>
<td class=cellinside>5.58e-02
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008143">abnormal dendrite morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.57e-04
</td>
<td class=cellinside>8.82e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010144">abnormal tumor vascularization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.02e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004090">abnormal sarcomere morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.75e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011428">mesangial cell hypoplasia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.75e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008137">absent podocytes</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.14e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0006254">thin cerebral cortex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.87e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005017">decreased B cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011854">cerebral edema</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.72e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008428">abnormal spatial working memory</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.87e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000260">abnormal angiogenesis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005502">abnormal renal/urinary system physiology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.35e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011339">abnormal glomerular mesangium morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.02e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005602">decreased angiogenesis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.10e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005402">abnormal action potential</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004753">abnormal miniature excitatory postsynaptic currents</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011003">reduced AMPA-mediated synaptic currents</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.30e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002061">abnormal aggression-related behavior</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.30e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009922">increased transitional stage T1 B cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.30e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000794">abnormal parietal lobe morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004757">abnormal distal convoluted tubule morphology</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 23 09:46:37 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>