<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod324</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod324</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>39 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10973">10973</a></p>
</td>
<td class=cellinside>ASCC3
</td>
<td class=cellinside>activating signal cointegrator 1 complex subunit 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=295334">295334</a></p>
</td>
<td class=cellinside>BCAS2
</td>
<td class=cellinside>BCAS2, pre-mRNA processing factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8896">8896</a></p>
</td>
<td class=cellinside>BUD31
</td>
<td class=cellinside>BUD31 homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=83938">83938</a></p>
</td>
<td class=cellinside>C10orf11
</td>
<td class=cellinside>leucine rich melanocyte differentiation associated
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=151903">151903</a></p>
</td>
<td class=cellinside>CCDC12
</td>
<td class=cellinside>coiled-coil domain containing 12
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55702">55702</a></p>
</td>
<td class=cellinside>CCDC94
</td>
<td class=cellinside>YJU2 splicing factor homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=988">988</a></p>
</td>
<td class=cellinside>CDC5L
</td>
<td class=cellinside>cell division cycle 5 like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51340">51340</a></p>
</td>
<td class=cellinside>CRNKL1
</td>
<td class=cellinside>crooked neck pre-mRNA splicing factor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57703">57703</a></p>
</td>
<td class=cellinside>CWC22
</td>
<td class=cellinside>CWC22 spliceosome associated protein homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=360613">360613</a></p>
</td>
<td class=cellinside>CWC25
</td>
<td class=cellinside>CWC25 spliceosome-associated protein homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1665">1665</a></p>
</td>
<td class=cellinside>DHX15
</td>
<td class=cellinside>DEAH-box helicase 15
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8449">8449</a></p>
</td>
<td class=cellinside>DHX16
</td>
<td class=cellinside>DEAH-box helicase 16
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54505">54505</a></p>
</td>
<td class=cellinside>DHX29
</td>
<td class=cellinside>DExH-box helicase 29
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=22907">22907</a></p>
</td>
<td class=cellinside>DHX30
</td>
<td class=cellinside>DExH-box helicase 30
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55760">55760</a></p>
</td>
<td class=cellinside>DHX32
</td>
<td class=cellinside>DEAH-box helicase 32 (putative)
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56919">56919</a></p>
</td>
<td class=cellinside>DHX33
</td>
<td class=cellinside>DEAH-box helicase 33
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9704">9704</a></p>
</td>
<td class=cellinside>DHX34
</td>
<td class=cellinside>DExH-box helicase 34
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=60625">60625</a></p>
</td>
<td class=cellinside>DHX35
</td>
<td class=cellinside>DEAH-box helicase 35
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=170506">170506</a></p>
</td>
<td class=cellinside>DHX36
</td>
<td class=cellinside>DEAH-box helicase 36
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79665">79665</a></p>
</td>
<td class=cellinside>DHX40
</td>
<td class=cellinside>DEAH-box helicase 40
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=90957">90957</a></p>
</td>
<td class=cellinside>DHX57
</td>
<td class=cellinside>DExH-box helicase 57
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1659">1659</a></p>
</td>
<td class=cellinside>DHX8
</td>
<td class=cellinside>DEAH-box helicase 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=165545">165545</a></p>
</td>
<td class=cellinside>DQX1
</td>
<td class=cellinside>DEAQ-box RNA dependent ATPase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=113510">113510</a></p>
</td>
<td class=cellinside>HELQ
</td>
<td class=cellinside>helicase, POLQ like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=164045">164045</a></p>
</td>
<td class=cellinside>HFM1
</td>
<td class=cellinside>HFM1, ATP dependent DNA helicase homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57461">57461</a></p>
</td>
<td class=cellinside>ISY1
</td>
<td class=cellinside>ISY1 splicing factor homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=100534599">100534599</a></p>
</td>
<td class=cellinside>ISY1-RAB43
</td>
<td class=cellinside>ISY1-RAB43 readthrough
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5356">5356</a></p>
</td>
<td class=cellinside>PLRG1
</td>
<td class=cellinside>pleiotropic regulator 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8559">8559</a></p>
</td>
<td class=cellinside>PRPF18
</td>
<td class=cellinside>pre-mRNA processing factor 18
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=27339">27339</a></p>
</td>
<td class=cellinside>PRPF19
</td>
<td class=cellinside>pre-mRNA processing factor 19
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55696">55696</a></p>
</td>
<td class=cellinside>RBM22
</td>
<td class=cellinside>RNA binding motif protein 22
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=163859">163859</a></p>
</td>
<td class=cellinside>SDE2
</td>
<td class=cellinside>SDE2 telomere maintenance homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=22938">22938</a></p>
</td>
<td class=cellinside>SNW1
</td>
<td class=cellinside>SNW domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=222183">222183</a></p>
</td>
<td class=cellinside>SRRM3
</td>
<td class=cellinside>serine/arginine repetitive matrix 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=170933">170933</a></p>
</td>
<td class=cellinside>SYF2
</td>
<td class=cellinside>SYF2 pre-mRNA-splicing factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=24144">24144</a></p>
</td>
<td class=cellinside>TFIP11
</td>
<td class=cellinside>tuftelin interacting protein 11
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56949">56949</a></p>
</td>
<td class=cellinside>XAB2
</td>
<td class=cellinside>XPA binding protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64848">64848</a></p>
</td>
<td class=cellinside>YTHDC2
</td>
<td class=cellinside>YTH domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51538">51538</a></p>
</td>
<td class=cellinside>ZCCHC17
</td>
<td class=cellinside>zinc finger CCHC-type containing 17
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>5.58e-01                                                                                                                    </td>
<td class=cellinside>1.21e-35                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0034459">ATP-dependent 3'-5' RNA helicase activity</a></p></td>
<td class=cellinside>molecular_function                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>3.62e-01                                                                                                                    </td>
<td class=cellinside>3.92e-22                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000974">Prp19 complex</a></p>                            </td>
<td class=cellinside>cellular_component                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>1.70e-01                                                                                                                    </td>
<td class=cellinside>3.09e-26                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071007">U2-type catalytic step 2 spliceosome</a></p>     </td>
<td class=cellinside>cellular_component                                                                                                          </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod324_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000375">RNA splicing, via transesterification reactions</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008380">RNA splicing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016071">mRNA metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006397">mRNA processing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000398">mRNA splicing, via spliceosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071013">catalytic step 2 spliceosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030529">intracellular ribonucleoprotein complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005681">spliceosomal complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1990904">ribonucleoprotein complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000974">Prp19 complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016887">ATPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0017111">nucleoside-triphosphatase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008026">ATP-dependent helicase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004386">helicase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0042623">ATPase activity, coupled</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0003724">RNA helicase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008186">RNA-dependent ATPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004004">ATP-dependent RNA helicase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.69e-11
</td>
<td class=cellinside>9.30e-09
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071014">post-mRNA release spliceosomal complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.40e-10
</td>
<td class=cellinside>7.07e-08
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071012">catalytic step 1 spliceosome</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>3.37e-03
</td>
<td class=cellinside>3.31e-01
</td>
<td class=cellinside>FORMATION OF TRANSCRIPTION COUPLED NER TC NER REPAIR COMPLEX
</td></tr>
 
<tr> 
<td class=cellinside>3.38e-03
</td>
<td class=cellinside>3.32e-01
</td>
<td class=cellinside>TRANSCRIPTION COUPLED NER TC NER
</td></tr>
 
<tr> 
<td class=cellinside>8.33e-03
</td>
<td class=cellinside>6.93e-01
</td>
<td class=cellinside>NUCLEOTIDE EXCISION REPAIR
</td></tr>
 
<tr> 
<td class=cellinside>8.64e-03
</td>
<td class=cellinside>6.53e-01
</td>
<td class=cellinside>FORMATION OF TRANSCRIPTION COUPLED NER TC NER REPAIR COMPLEX
</td></tr>
 
<tr> 
<td class=cellinside>8.67e-03
</td>
<td class=cellinside>6.55e-01
</td>
<td class=cellinside>TRANSCRIPTION COUPLED NER TC NER
</td></tr>
 
<tr> 
<td class=cellinside>9.00e-03
</td>
<td class=cellinside>6.75e-01
</td>
<td class=cellinside>NOTCH HLH TRANSCRIPTION PATHWAY
</td></tr>
 
<tr> 
<td class=cellinside>1.03e-02
</td>
<td class=cellinside>8.15e-01
</td>
<td class=cellinside>NOTCH HLH TRANSCRIPTION PATHWAY
</td></tr>
 
<tr> 
<td class=cellinside>1.42e-02
</td>
<td class=cellinside>9.73e-01
</td>
<td class=cellinside>NOTCH1 INTRACELLULAR DOMAIN REGULATES TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>1.67e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>NUCLEOTIDE EXCISION REPAIR
</td></tr>
 
<tr> 
<td class=cellinside>1.68e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PRE NOTCH TRANSCRIPTION AND TRANSLATION
</td></tr>
 
<tr> 
<td class=cellinside>1.97e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNALING BY NOTCH1
</td></tr>
 
<tr> 
<td class=cellinside>2.02e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>NOTCH1 INTRACELLULAR DOMAIN REGULATES TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>2.18e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PRE NOTCH EXPRESSION AND PROCESSING
</td></tr>
 
<tr> 
<td class=cellinside>2.26e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PRE NOTCH TRANSCRIPTION AND TRANSLATION
</td></tr>
 
<tr> 
<td class=cellinside>2.27e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PRE NOTCH EXPRESSION AND PROCESSING
</td></tr>
 
<tr> 
<td class=cellinside>3.03e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNALING BY NOTCH1
</td></tr>
 
<tr> 
<td class=cellinside>3.34e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNALING BY NOTCH
</td></tr>
 
<tr> 
<td class=cellinside>4.67e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNALING BY NOTCH
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.88e-04
</td>
<td class=cellinside>3.35e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001126">abnormal ovary morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.68e-04
</td>
<td class=cellinside>8.33e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011094">complete embryonic lethality before implantation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.87e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011751">abnormal X-Y chromosome synapsis during male meiosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.74e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010288">increased gland tumor incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.75e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008000">increased ovary tumor incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002041">increased pituitary adenoma incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004833">ovary atrophy</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008871">abnormal ovarian follicle number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002680">decreased corpora lutea number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008966">abnormal chiasmata formation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008039">increased NK T cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.49e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001931">abnormal oogenesis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.50e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001132">absent mature ovarian follicles</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.15e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000352">decreased cell proliferation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004357">long tibia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009452">abnormal synaptonemal complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001930">abnormal meiosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.36e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002016">ovary cysts</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.06e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010299">increased mammary gland tumor incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009451">abnormal chromosome pairing during meiosis</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 23 10:00:26 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>