<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
<p class='character'><a href="../index.html">Back to main page</a></p>

 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod184</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod184</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>50 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=326">326</a></p>
</td>
<td class=cellinside>AIRE
</td>
<td class=cellinside>autoimmune regulator
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9070">9070</a></p>
</td>
<td class=cellinside>ASH2L
</td>
<td class=cellinside>ASH2 like, histone lysine methyltransferase complex subunit
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80816">80816</a></p>
</td>
<td class=cellinside>ASXL3
</td>
<td class=cellinside>ASXL transcriptional regulator 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10951">10951</a></p>
</td>
<td class=cellinside>CBX1
</td>
<td class=cellinside>chromobox 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11335">11335</a></p>
</td>
<td class=cellinside>CBX3
</td>
<td class=cellinside>chromobox 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=30827">30827</a></p>
</td>
<td class=cellinside>CXXC1
</td>
<td class=cellinside>CXXC finger protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=203522">203522</a></p>
</td>
<td class=cellinside>DDX26B
</td>
<td class=cellinside>integrator complex subunit 6 like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11083">11083</a></p>
</td>
<td class=cellinside>DIDO1
</td>
<td class=cellinside>death inducer-obliterator 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84444">84444</a></p>
</td>
<td class=cellinside>DOT1L
</td>
<td class=cellinside>DOT1 like histone lysine methyltransferase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2145">2145</a></p>
</td>
<td class=cellinside>EZH1
</td>
<td class=cellinside>enhancer of zeste 1 polycomb repressive complex 2 subunit
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2146">2146</a></p>
</td>
<td class=cellinside>EZH2
</td>
<td class=cellinside>enhancer of zeste 2 polycomb repressive complex 2 subunit
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=285596">285596</a></p>
</td>
<td class=cellinside>FAM153A
</td>
<td class=cellinside>family with sequence similarity 153 member A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79709">79709</a></p>
</td>
<td class=cellinside>GLT25D1
</td>
<td class=cellinside>collagen beta(1-O)galactosyltransferase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23127">23127</a></p>
</td>
<td class=cellinside>GLT25D2
</td>
<td class=cellinside>collagen beta(1-O)galactosyltransferase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8350">8350</a></p>
</td>
<td class=cellinside>HIST1H3A
</td>
<td class=cellinside>histone cluster 1 H3 family member a
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8290">8290</a></p>
</td>
<td class=cellinside>HIST3H3
</td>
<td class=cellinside>histone cluster 3 H3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=221656">221656</a></p>
</td>
<td class=cellinside>KDM1B
</td>
<td class=cellinside>lysine demethylase 1B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9682">9682</a></p>
</td>
<td class=cellinside>KDM4A
</td>
<td class=cellinside>lysine demethylase 4A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23030">23030</a></p>
</td>
<td class=cellinside>KDM4B
</td>
<td class=cellinside>lysine demethylase 4B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23081">23081</a></p>
</td>
<td class=cellinside>KDM4C
</td>
<td class=cellinside>lysine demethylase 4C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9757">9757</a></p>
</td>
<td class=cellinside>MLL2
</td>
<td class=cellinside>lysine methyltransferase 2B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=58508">58508</a></p>
</td>
<td class=cellinside>MLL3
</td>
<td class=cellinside>lysine methyltransferase 2C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55904">55904</a></p>
</td>
<td class=cellinside>MLL5
</td>
<td class=cellinside>lysine methyltransferase 2E
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64324">64324</a></p>
</td>
<td class=cellinside>NSD1
</td>
<td class=cellinside>nuclear receptor binding SET domain protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51131">51131</a></p>
</td>
<td class=cellinside>PHF11
</td>
<td class=cellinside>PHD finger protein 11
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51533">51533</a></p>
</td>
<td class=cellinside>PHF7
</td>
<td class=cellinside>PHD finger protein 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5351">5351</a></p>
</td>
<td class=cellinside>PLOD1
</td>
<td class=cellinside>procollagen-lysine,2-oxoglutarate 5-dioxygenase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5352">5352</a></p>
</td>
<td class=cellinside>PLOD2
</td>
<td class=cellinside>procollagen-lysine,2-oxoglutarate 5-dioxygenase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8985">8985</a></p>
</td>
<td class=cellinside>PLOD3
</td>
<td class=cellinside>procollagen-lysine,2-oxoglutarate 5-dioxygenase 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26040">26040</a></p>
</td>
<td class=cellinside>SETBP1
</td>
<td class=cellinside>SET binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9739">9739</a></p>
</td>
<td class=cellinside>SETD1A
</td>
<td class=cellinside>SET domain containing 1A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23067">23067</a></p>
</td>
<td class=cellinside>SETD1B
</td>
<td class=cellinside>SET domain containing 1B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80854">80854</a></p>
</td>
<td class=cellinside>SETD7
</td>
<td class=cellinside>SET domain containing lysine methyltransferase 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9869">9869</a></p>
</td>
<td class=cellinside>SETDB1
</td>
<td class=cellinside>SET domain bifurcated 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=83852">83852</a></p>
</td>
<td class=cellinside>SETDB2
</td>
<td class=cellinside>SET domain bifurcated 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6419">6419</a></p>
</td>
<td class=cellinside>SETMAR
</td>
<td class=cellinside>SET domain and mariner transposase fusion gene
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=150572">150572</a></p>
</td>
<td class=cellinside>SMYD1
</td>
<td class=cellinside>SET and MYND domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56950">56950</a></p>
</td>
<td class=cellinside>SMYD2
</td>
<td class=cellinside>SET and MYND domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64754">64754</a></p>
</td>
<td class=cellinside>SMYD3
</td>
<td class=cellinside>SET and MYND domain containing 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=114826">114826</a></p>
</td>
<td class=cellinside>SMYD4
</td>
<td class=cellinside>SET and MYND domain containing 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10322">10322</a></p>
</td>
<td class=cellinside>SMYD5
</td>
<td class=cellinside>SMYD family member 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6839">6839</a></p>
</td>
<td class=cellinside>SUV39H1
</td>
<td class=cellinside>suppressor of variegation 3-9 homolog 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79723">79723</a></p>
</td>
<td class=cellinside>SUV39H2
</td>
<td class=cellinside>suppressor of variegation 3-9 homolog 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51111">51111</a></p>
</td>
<td class=cellinside>SUV420H1
</td>
<td class=cellinside>lysine methyltransferase 5B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84787">84787</a></p>
</td>
<td class=cellinside>SUV420H2
</td>
<td class=cellinside>lysine methyltransferase 5C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=170082">170082</a></p>
</td>
<td class=cellinside>TCEANC
</td>
<td class=cellinside>transcription elongation factor A N-terminal and central domain containing
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5728">5728</a></p>
</td>
<td class=cellinside>TEP1
</td>
<td class=cellinside>phosphatase and tensin homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1787">1787</a></p>
</td>
<td class=cellinside>TRDMT1
</td>
<td class=cellinside>tRNA aspartic acid methyltransferase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=7468">7468</a></p>
</td>
<td class=cellinside>WHSC1
</td>
<td class=cellinside>nuclear receptor binding SET domain protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54904">54904</a></p>
</td>
<td class=cellinside>WHSC1L1
</td>
<td class=cellinside>nuclear receptor binding SET domain protein 3
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>3.35e-01                                                                                                                                                              </td>
<td class=cellinside>7.75e-09                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008475">procollagen-lysine 5-dioxygenase activity</a></p>                                          </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>3.34e-01                                                                                                                                                              </td>
<td class=cellinside>7.75e-09                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046947">hydroxylysine biosynthetic process</a></p>                                                 </td>
<td class=cellinside>biological_process                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>2.24e-01                                                                                                                                                              </td>
<td class=cellinside>5.01e-32                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0018024">histone-lysine N-methyltransferase activity</a></p>                                        </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>8.14e-02                                                                                                                                                              </td>
<td class=cellinside>9.21e-07                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0070544">histone H3-K36 demethylation</a></p>                                                       </td>
<td class=cellinside>biological_process                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>6.27e-02                                                                                                                                                              </td>
<td class=cellinside>1.54e-07                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046975">histone methyltransferase activity (H3-K36 specific)</a></p>                               </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>4.78e-02                                                                                                                                                              </td>
<td class=cellinside>2.04e-09                                                                                                                                                              </td>
<td class=cellinside>Reactome                                                                                                                                                              </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-3214842&species=Homo+sapiens&species=Entries+without+species&cluster=true">HDMs demethylate histones</a></p>      </td>
<td class=cellinside>Chromatin organization                                                                                                                                                </td> </tr>
 <tr> 
<td class=cellinside>3.81e-02                                                                                                                                                              </td>
<td class=cellinside>1.02e-08                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0048188">Set1C/COMPASS complex</a></p>                                                              </td>
<td class=cellinside>cellular_component                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>3.73e-02                                                                                                                                                              </td>
<td class=cellinside>3.44e-28                                                                                                                                                              </td>
<td class=cellinside>Reactome                                                                                                                                                              </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-3214841&species=Homo+sapiens&species=Entries+without+species&cluster=true">PKMTs methylate histone lysines</a></p></td>
<td class=cellinside>Chromatin organization                                                                                                                                                </td> </tr>
 <tr> 
<td class=cellinside>3.35e-02                                                                                                                                                              </td>
<td class=cellinside>1.26e-06                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0042054">histone methyltransferase activity</a></p>                                                 </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>7.54e-03                                                                                                                                                              </td>
<td class=cellinside>3.10e-04                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0090734">site of DNA damage</a></p>                                                                 </td>
<td class=cellinside>cellular_component                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>5.94e-03                                                                                                                                                              </td>
<td class=cellinside>1.26e-06                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0051864">histone demethylase activity (H3-K36 specific)</a></p>                                     </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>3.90e-03                                                                                                                                                              </td>
<td class=cellinside>2.78e-06                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032454">histone demethylase activity (H3-K9 specific)</a></p>                                      </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod184_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032259">methylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016569">covalent chromatin modification</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006325">chromatin organization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0034968">histone lysine methylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016570">histone modification</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016571">histone methylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006479">protein methylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0018022">peptidyl-lysine methylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0018205">peptidyl-lysine modification</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000785">chromatin</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008168">methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008757">S-adenosylmethionine-dependent methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008170">N-methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008276">protein methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0042054">histone methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0018024">histone-lysine N-methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016278">lysine N-methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016279">protein-lysine N-methyltransferase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.37e-10
</td>
<td class=cellinside>1.89e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0018027">peptidyl-lysine dimethylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.16e-09
</td>
<td class=cellinside>1.33e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035097">histone methyltransferase complex</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.70e-07
</td>
<td class=cellinside>4.28e-05
</td>
<td class=cellinside>COLLAGEN FORMATION
</td></tr>
 
<tr> 
<td class=cellinside>1.03e-06
</td>
<td class=cellinside>1.78e-04
</td>
<td class=cellinside>COLLAGEN FORMATION
</td></tr>
 
<tr> 
<td class=cellinside>1.23e-06
</td>
<td class=cellinside>2.75e-04
</td>
<td class=cellinside>EXTRACELLULAR MATRIX ORGANIZATION
</td></tr>
 
<tr> 
<td class=cellinside>6.40e-06
</td>
<td class=cellinside>9.87e-04
</td>
<td class=cellinside>EXTRACELLULAR MATRIX ORGANIZATION
</td></tr>
 
<tr> 
<td class=cellinside>8.00e-04
</td>
<td class=cellinside>9.68e-02
</td>
<td class=cellinside>MEIOSIS
</td></tr>
 
<tr> 
<td class=cellinside>8.08e-04
</td>
<td class=cellinside>8.30e-02
</td>
<td class=cellinside>MEIOSIS
</td></tr>
 
<tr> 
<td class=cellinside>2.22e-03
</td>
<td class=cellinside>2.04e-01
</td>
<td class=cellinside>MEIOTIC SYNAPSIS
</td></tr>
 
<tr> 
<td class=cellinside>2.32e-03
</td>
<td class=cellinside>2.42e-01
</td>
<td class=cellinside>RNA POL I TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>2.57e-03
</td>
<td class=cellinside>2.64e-01
</td>
<td class=cellinside>MEIOTIC SYNAPSIS
</td></tr>
 
<tr> 
<td class=cellinside>2.70e-03
</td>
<td class=cellinside>2.42e-01
</td>
<td class=cellinside>RNA POL I TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>5.37e-03
</td>
<td class=cellinside>4.85e-01
</td>
<td class=cellinside>MEIOTIC RECOMBINATION
</td></tr>
 
<tr> 
<td class=cellinside>6.99e-03
</td>
<td class=cellinside>5.47e-01
</td>
<td class=cellinside>MEIOTIC RECOMBINATION
</td></tr>
 
<tr> 
<td class=cellinside>7.25e-03
</td>
<td class=cellinside>6.18e-01
</td>
<td class=cellinside>CHROMOSOME MAINTENANCE
</td></tr>
 
<tr> 
<td class=cellinside>7.60e-03
</td>
<td class=cellinside>5.86e-01
</td>
<td class=cellinside>RNA POL I RNA POL III AND MITOCHONDRIAL TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>8.53e-03
</td>
<td class=cellinside>6.45e-01
</td>
<td class=cellinside>CHROMOSOME MAINTENANCE
</td></tr>
 
<tr> 
<td class=cellinside>1.20e-02
</td>
<td class=cellinside>9.20e-01
</td>
<td class=cellinside>PACKAGING OF TELOMERE ENDS
</td></tr>
 
<tr> 
<td class=cellinside>1.27e-02
</td>
<td class=cellinside>8.92e-01
</td>
<td class=cellinside>PACKAGING OF TELOMERE ENDS
</td></tr>
 
<tr> 
<td class=cellinside>1.43e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RNA POL I RNA POL III AND MITOCHONDRIAL TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>2.88e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TELOMERE MAINTENANCE
</td></tr>
 
<tr> 
<td class=cellinside>3.18e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TELOMERE MAINTENANCE
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.95e-04
</td>
<td class=cellinside>3.43e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002023">increased B cell derived lymphoma incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.05e-04
</td>
<td class=cellinside>7.38e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010770">preweaning lethality</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.43e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000753">paralysis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.63e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011097">complete embryonic lethality between somite formation and embryo turning</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.12e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008438">abnormal cutaneous collagen fibril morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008762">embryonic lethality</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.77e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0013367">parotid gland inflammation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.77e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003504">thyroid inflammation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009332">abnormal splenocyte morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008862">asymmetric snout</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.64e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009540">absent Hassall's corpuscle</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.64e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003608">prostate gland inflammation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.65e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003726">decreased autoantibody level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.65e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008879">submandibular gland inflammation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.21e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004174">abnormal spine curvature</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.52e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008241">abnormal metallophilic macrophage morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.12e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004024">aneuploidy</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.69e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008045">decreased NK cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.35e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005515">uveitis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.39e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0020220">decreased tear production</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 30 17:28:53 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>