<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
<p class='character'><a href="../index.html">Back to main page</a></p>

 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod60</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                        </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod60</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>50 genes                  </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9744">9744</a></p>
</td>
<td class=cellinside>ACAP1
</td>
<td class=cellinside>ArfGAP with coiled-coil, ankyrin repeat and PH domains 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23527">23527</a></p>
</td>
<td class=cellinside>ACAP2
</td>
<td class=cellinside>ArfGAP with coiled-coil, ankyrin repeat and PH domains 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=116983">116983</a></p>
</td>
<td class=cellinside>ACAP3
</td>
<td class=cellinside>ArfGAP with coiled-coil, ankyrin repeat and PH domains 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11033">11033</a></p>
</td>
<td class=cellinside>ADAP1
</td>
<td class=cellinside>ArfGAP with dual PH domains 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55803">55803</a></p>
</td>
<td class=cellinside>ADAP2
</td>
<td class=cellinside>ArfGAP with dual PH domains 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=375">375</a></p>
</td>
<td class=cellinside>ARF1
</td>
<td class=cellinside>ADP ribosylation factor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=377">377</a></p>
</td>
<td class=cellinside>ARF3
</td>
<td class=cellinside>ADP ribosylation factor 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=378">378</a></p>
</td>
<td class=cellinside>ARF4
</td>
<td class=cellinside>ADP ribosylation factor 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=381">381</a></p>
</td>
<td class=cellinside>ARF5
</td>
<td class=cellinside>ADP ribosylation factor 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=382">382</a></p>
</td>
<td class=cellinside>ARF6
</td>
<td class=cellinside>ADP ribosylation factor 6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26286">26286</a></p>
</td>
<td class=cellinside>ARFGAP1
</td>
<td class=cellinside>ADP ribosylation factor GTPase activating protein 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84364">84364</a></p>
</td>
<td class=cellinside>ARFGAP2
</td>
<td class=cellinside>ADP ribosylation factor GTPase activating protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26286">26286</a></p>
</td>
<td class=cellinside>ARFGAP3
</td>
<td class=cellinside>ADP ribosylation factor GTPase activating protein 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10565">10565</a></p>
</td>
<td class=cellinside>ARFGEF1
</td>
<td class=cellinside>ADP ribosylation factor guanine nucleotide exchange factor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10564">10564</a></p>
</td>
<td class=cellinside>ARFGEF2
</td>
<td class=cellinside>ADP ribosylation factor guanine nucleotide exchange factor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=400">400</a></p>
</td>
<td class=cellinside>ARL1
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=115761">115761</a></p>
</td>
<td class=cellinside>ARL11
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 11
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80117">80117</a></p>
</td>
<td class=cellinside>ARL14
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 14
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54622">54622</a></p>
</td>
<td class=cellinside>ARL15
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 15
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51326">51326</a></p>
</td>
<td class=cellinside>ARL17A
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 17A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=100506084">100506084</a></p>
</td>
<td class=cellinside>ARL17B
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 17B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10124">10124</a></p>
</td>
<td class=cellinside>ARL4A
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 4A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10123">10123</a></p>
</td>
<td class=cellinside>ARL4C
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 4C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=379">379</a></p>
</td>
<td class=cellinside>ARL4D
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 4D
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26225">26225</a></p>
</td>
<td class=cellinside>ARL5A
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 5A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=221079">221079</a></p>
</td>
<td class=cellinside>ARL5B
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 5B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=390790">390790</a></p>
</td>
<td class=cellinside>ARL5C
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 5C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=379">379</a></p>
</td>
<td class=cellinside>ARL6
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 4D
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=132946">132946</a></p>
</td>
<td class=cellinside>ARL9
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 9
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8853">8853</a></p>
</td>
<td class=cellinside>ASAP2
</td>
<td class=cellinside>ArfGAP with SH3 domain, ankyrin repeat and PH domain 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55616">55616</a></p>
</td>
<td class=cellinside>ASAP3
</td>
<td class=cellinside>ArfGAP with SH3 domain, ankyrin repeat and PH domain 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26958">26958</a></p>
</td>
<td class=cellinside>COPG2
</td>
<td class=cellinside>coatomer protein complex subunit gamma 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9267">9267</a></p>
</td>
<td class=cellinside>CYTH1
</td>
<td class=cellinside>cytohesin 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9266">9266</a></p>
</td>
<td class=cellinside>CYTH2
</td>
<td class=cellinside>cytohesin 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9265">9265</a></p>
</td>
<td class=cellinside>CYTH3
</td>
<td class=cellinside>cytohesin 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=27128">27128</a></p>
</td>
<td class=cellinside>CYTH4
</td>
<td class=cellinside>cytohesin 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26269">26269</a></p>
</td>
<td class=cellinside>FBXO8
</td>
<td class=cellinside>F-box protein 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80142">80142</a></p>
</td>
<td class=cellinside>GBF1
</td>
<td class=cellinside>prostaglandin E synthase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9922">9922</a></p>
</td>
<td class=cellinside>IQSEC1
</td>
<td class=cellinside>IQ motif and Sec7 domain 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23096">23096</a></p>
</td>
<td class=cellinside>IQSEC2
</td>
<td class=cellinside>IQ motif and Sec7 domain 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=440073">440073</a></p>
</td>
<td class=cellinside>IQSEC3
</td>
<td class=cellinside>IQ motif and Sec7 domain 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10945">10945</a></p>
</td>
<td class=cellinside>KDELR1
</td>
<td class=cellinside>KDEL endoplasmic reticulum protein retention receptor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11014">11014</a></p>
</td>
<td class=cellinside>KDELR2
</td>
<td class=cellinside>KDEL endoplasmic reticulum protein retention receptor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11015">11015</a></p>
</td>
<td class=cellinside>KDELR3
</td>
<td class=cellinside>KDEL endoplasmic reticulum protein retention receptor 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84249">84249</a></p>
</td>
<td class=cellinside>PSD2
</td>
<td class=cellinside>pleckstrin and Sec7 domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23362">23362</a></p>
</td>
<td class=cellinside>PSD3
</td>
<td class=cellinside>pleckstrin and Sec7 domain containing 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23550">23550</a></p>
</td>
<td class=cellinside>PSD4
</td>
<td class=cellinside>pleckstrin and Sec7 domain containing 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=60682">60682</a></p>
</td>
<td class=cellinside>SMAP1
</td>
<td class=cellinside>small ArfGAP 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10902">10902</a></p>
</td>
<td class=cellinside>SMAP2
</td>
<td class=cellinside>bromodomain containing 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=25972">25972</a></p>
</td>
<td class=cellinside>UNC50
</td>
<td class=cellinside>unc-50 inner nuclear membrane RNA binding protein
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>3.93e-01                                                                                                                                                                            </td>
<td class=cellinside>7.31e-35                                                                                                                                                                            </td>
<td class=cellinside>GO                                                                                                                                                                                  </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032012">regulation of ARF protein signal transduction</a></p>                                                    </td>
<td class=cellinside>biological_process                                                                                                                                                                  </td> </tr>
 <tr> 
<td class=cellinside>2.52e-01                                                                                                                                                                            </td>
<td class=cellinside>3.10e-36                                                                                                                                                                            </td>
<td class=cellinside>GO                                                                                                                                                                                  </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005086">ARF guanyl-nucleotide exchange factor activity</a></p>                                                   </td>
<td class=cellinside>molecular_function                                                                                                                                                                  </td> </tr>
 <tr> 
<td class=cellinside>1.21e-01                                                                                                                                                                            </td>
<td class=cellinside>1.49e-06                                                                                                                                                                            </td>
<td class=cellinside>GO                                                                                                                                                                                  </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006621">protein retention in ER lumen</a></p>                                                                    </td>
<td class=cellinside>biological_process                                                                                                                                                                  </td> </tr>
 <tr> 
<td class=cellinside>8.30e-02                                                                                                                                                                            </td>
<td class=cellinside>1.01e-31                                                                                                                                                                            </td>
<td class=cellinside>GO                                                                                                                                                                                  </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016192">vesicle-mediated transport</a></p>                                                                       </td>
<td class=cellinside>biological_process                                                                                                                                                                  </td> </tr>
 <tr> 
<td class=cellinside>4.13e-02                                                                                                                                                                            </td>
<td class=cellinside>2.41e-18                                                                                                                                                                            </td>
<td class=cellinside>Reactome                                                                                                                                                                            </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-6807878&species=Homo+sapiens&species=Entries+without+species&cluster=true">COPI-mediated anterograde transport</a></p>          </td>
<td class=cellinside>Metabolism of proteins#Vesicle-mediated transport                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>3.55e-02                                                                                                                                                                            </td>
<td class=cellinside>2.41e-18                                                                                                                                                                            </td>
<td class=cellinside>Reactome                                                                                                                                                                            </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-6811434&species=Homo+sapiens&species=Entries+without+species&cluster=true">COPI-dependent Golgi-to-ER retrograde traffic</a></p></td>
<td class=cellinside>Vesicle-mediated transport                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>2.08e-02                                                                                                                                                                            </td>
<td class=cellinside>4.51e-09                                                                                                                                                                            </td>
<td class=cellinside>GO                                                                                                                                                                                  </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006893">Golgi to plasma membrane transport</a></p>                                                               </td>
<td class=cellinside>biological_process                                                                                                                                                                  </td> </tr>
 <tr> 
<td class=cellinside>6.86e-03                                                                                                                                                                            </td>
<td class=cellinside>2.25e-16                                                                                                                                                                            </td>
<td class=cellinside>GO                                                                                                                                                                                  </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006890">retrograde vesicle-mediated transport, Golgi to ER</a></p>                                               </td>
<td class=cellinside>biological_process                                                                                                                                                                  </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod60_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0051056">regulation of small GTPase mediated signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043087">regulation of GTPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046578">regulation of Ras protein signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032012">regulation of ARF protein signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043547">positive regulation of GTPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0007264">small GTPase mediated signal transduction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0048193">Golgi vesicle transport</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006890">retrograde vesicle-mediated transport, Golgi to ER</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000139">Golgi membrane</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005086">ARF guanyl-nucleotide exchange factor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005085">guanyl-nucleotide exchange factor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0001883">purine nucleoside binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005525">GTP binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032549">ribonucleoside binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032561">guanyl ribonucleotide binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0001882">nucleoside binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.73e-10
</td>
<td class=cellinside>3.15e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005096">GTPase activator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.17e-10
</td>
<td class=cellinside>3.37e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008047">enzyme activator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.48e-09
</td>
<td class=cellinside>7.64e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030695">GTPase regulator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.71e-07
</td>
<td class=cellinside>9.27e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0033365">protein localization to organelle</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.02e-07
</td>
<td class=cellinside>1.99e-05
</td>
<td class=cellinside>COPI MEDIATED TRANSPORT
</td></tr>
 
<tr> 
<td class=cellinside>5.72e-06
</td>
<td class=cellinside>1.14e-03
</td>
<td class=cellinside>COPI MEDIATED TRANSPORT
</td></tr>
 
<tr> 
<td class=cellinside>2.60e-04
</td>
<td class=cellinside>3.01e-02
</td>
<td class=cellinside>SYNTHESIS OF PIPS AT THE GOLGI MEMBRANE
</td></tr>
 
<tr> 
<td class=cellinside>4.37e-04
</td>
<td class=cellinside>4.80e-02
</td>
<td class=cellinside>MEMBRANE TRAFFICKING
</td></tr>
 
<tr> 
<td class=cellinside>6.14e-04
</td>
<td class=cellinside>7.70e-02
</td>
<td class=cellinside>SYNTHESIS OF PIPS AT THE GOLGI MEMBRANE
</td></tr>
 
<tr> 
<td class=cellinside>2.34e-03
</td>
<td class=cellinside>2.14e-01
</td>
<td class=cellinside>PI METABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>2.84e-03
</td>
<td class=cellinside>2.53e-01
</td>
<td class=cellinside>ACTIVATION OF CHAPERONE GENES BY XBP1S
</td></tr>
 
<tr> 
<td class=cellinside>3.20e-03
</td>
<td class=cellinside>2.81e-01
</td>
<td class=cellinside>TRANS GOLGI NETWORK VESICLE BUDDING
</td></tr>
 
<tr> 
<td class=cellinside>4.14e-03
</td>
<td class=cellinside>3.91e-01
</td>
<td class=cellinside>PI METABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>4.48e-03
</td>
<td class=cellinside>4.17e-01
</td>
<td class=cellinside>MEMBRANE TRAFFICKING
</td></tr>
 
<tr> 
<td class=cellinside>5.23e-03
</td>
<td class=cellinside>4.74e-01
</td>
<td class=cellinside>ACTIVATION OF CHAPERONE GENES BY XBP1S
</td></tr>
 
<tr> 
<td class=cellinside>6.15e-03
</td>
<td class=cellinside>4.89e-01
</td>
<td class=cellinside>UNFOLDED PROTEIN RESPONSE
</td></tr>
 
<tr> 
<td class=cellinside>8.94e-03
</td>
<td class=cellinside>7.32e-01
</td>
<td class=cellinside>UNFOLDED PROTEIN RESPONSE
</td></tr>
 
<tr> 
<td class=cellinside>9.87e-03
</td>
<td class=cellinside>7.93e-01
</td>
<td class=cellinside>TRANS GOLGI NETWORK VESICLE BUDDING
</td></tr>
 
<tr> 
<td class=cellinside>1.13e-02
</td>
<td class=cellinside>8.10e-01
</td>
<td class=cellinside>DIABETES PATHWAYS
</td></tr>
 
<tr> 
<td class=cellinside>1.63e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>ASSOCIATION OF TRIC CCT WITH TARGET PROTEINS DURING BIOSYNTHESIS
</td></tr>
 
<tr> 
<td class=cellinside>2.00e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>LYSOSOME VESICLE BIOGENESIS
</td></tr>
 
<tr> 
<td class=cellinside>2.15e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>NEF MEDIATES DOWN MODULATION OF CELL SURFACE RECEPTORS BY RECRUITING THEM TO CLATHRIN ADAPTERS
</td></tr>
 
<tr> 
<td class=cellinside>2.88e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>NEF MEDIATES DOWN MODULATION OF CELL SURFACE RECEPTORS BY RECRUITING THEM TO CLATHRIN ADAPTERS
</td></tr>
 
<tr> 
<td class=cellinside>2.98e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PROTEIN FOLDING
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>4.36e-04
</td>
<td class=cellinside>5.6e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008283">small hippocampus</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.87e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008041">absent NK T cells</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.88e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002811">macrocytic anemia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.74e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008583">absent photoreceptor inner segment</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002200">abnormal brain ventricular system morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008581">disorganized photoreceptor inner segment</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009923">decreased transitional stage T1 B cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012246">abnormal hepatic cord morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.24e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010124">decreased bone mineral content</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.92e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003961">decreased lean body mass</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.33e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011069">abnormal brain ependyma motile cilium physiology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009926">decreased transitional stage T2 B cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.09e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011723">ectopic neuron</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004183">abnormal sympathetic nervous system physiology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003257">abnormal abdominal wall morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004677">truncated ribs</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005452">abnormal adipose tissue amount</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002026">increased leukemia incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.49e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002835">abnormal cranial suture morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.49e-02
</td>
<td class=cellinside>1.0e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005065">abnormal neutrophil morphology</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 30 17:22:38 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>