<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod298</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod298</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>43 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1820">1820</a></p>
</td>
<td class=cellinside>ARID3A
</td>
<td class=cellinside>AT-rich interaction domain 3A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10620">10620</a></p>
</td>
<td class=cellinside>ARID3B
</td>
<td class=cellinside>AT-rich interaction domain 3B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=138715">138715</a></p>
</td>
<td class=cellinside>ARID3C
</td>
<td class=cellinside>AT-rich interaction domain 3C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10865">10865</a></p>
</td>
<td class=cellinside>ARID5A
</td>
<td class=cellinside>AT-rich interaction domain 5A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=546">546</a></p>
</td>
<td class=cellinside>ATRX
</td>
<td class=cellinside>ATRX, chromatin remodeler
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11177">11177</a></p>
</td>
<td class=cellinside>BAZ1A
</td>
<td class=cellinside>bromodomain adjacent to zinc finger domain 1A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9031">9031</a></p>
</td>
<td class=cellinside>BAZ1B
</td>
<td class=cellinside>bromodomain adjacent to zinc finger domain 1B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11176">11176</a></p>
</td>
<td class=cellinside>BAZ2A
</td>
<td class=cellinside>bromodomain adjacent to zinc finger domain 2A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=29994">29994</a></p>
</td>
<td class=cellinside>BAZ2B
</td>
<td class=cellinside>bromodomain adjacent to zinc finger domain 2B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2186">2186</a></p>
</td>
<td class=cellinside>BPTF
</td>
<td class=cellinside>bromodomain PHD finger transcription factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6046">6046</a></p>
</td>
<td class=cellinside>BRD2
</td>
<td class=cellinside>bromodomain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8019">8019</a></p>
</td>
<td class=cellinside>BRD3
</td>
<td class=cellinside>bromodomain containing 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54014">54014</a></p>
</td>
<td class=cellinside>BRWD1
</td>
<td class=cellinside>bromodomain and WD repeat domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9044">9044</a></p>
</td>
<td class=cellinside>BTAF1
</td>
<td class=cellinside>B-TFIID TATA-box binding protein associated factor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=124773">124773</a></p>
</td>
<td class=cellinside>C17orf64
</td>
<td class=cellinside>chromosome 17 open reading frame 64
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1105">1105</a></p>
</td>
<td class=cellinside>CHD1
</td>
<td class=cellinside>chromodomain helicase DNA binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9557">9557</a></p>
</td>
<td class=cellinside>CHD1L
</td>
<td class=cellinside>chromodomain helicase DNA binding protein 1 like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1826">1826</a></p>
</td>
<td class=cellinside>CHD2
</td>
<td class=cellinside>DS cell adhesion molecule
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=288233">288233</a></p>
</td>
<td class=cellinside>CHD5
</td>
<td class=cellinside>tryptophan rich basic protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84181">84181</a></p>
</td>
<td class=cellinside>CHD6
</td>
<td class=cellinside>chromodomain helicase DNA binding protein 6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55636">55636</a></p>
</td>
<td class=cellinside>CHD7
</td>
<td class=cellinside>chromodomain helicase DNA binding protein 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=347918">347918</a></p>
</td>
<td class=cellinside>EP400NL
</td>
<td class=cellinside>EP400 pseudogene 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=375748">375748</a></p>
</td>
<td class=cellinside>ERCC6L2
</td>
<td class=cellinside>ERCC excision repair 6 like 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6596">6596</a></p>
</td>
<td class=cellinside>HLTF
</td>
<td class=cellinside>helicase like transcription factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10363">10363</a></p>
</td>
<td class=cellinside>HMG20A
</td>
<td class=cellinside>high mobility group 20A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10362">10362</a></p>
</td>
<td class=cellinside>HMG20B
</td>
<td class=cellinside>high mobility group 20B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=3148">3148</a></p>
</td>
<td class=cellinside>HMGB2
</td>
<td class=cellinside>high mobility group box 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=3149">3149</a></p>
</td>
<td class=cellinside>HMGB3
</td>
<td class=cellinside>high mobility group box 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=127540">127540</a></p>
</td>
<td class=cellinside>HMGB4
</td>
<td class=cellinside>high mobility group box 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10042">10042</a></p>
</td>
<td class=cellinside>HMGXB4
</td>
<td class=cellinside>HMG-box containing 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=126321">126321</a></p>
</td>
<td class=cellinside>MFSD12
</td>
<td class=cellinside>major facilitator superfamily domain containing 12
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23132">23132</a></p>
</td>
<td class=cellinside>RAD54L2
</td>
<td class=cellinside>RAD54 like 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=257218">257218</a></p>
</td>
<td class=cellinside>SHPRH
</td>
<td class=cellinside>SNF2 histone linker PHD RING helicase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6594">6594</a></p>
</td>
<td class=cellinside>SMARCA1
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily a, member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8467">8467</a></p>
</td>
<td class=cellinside>SMARCA5
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily a, member 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56916">56916</a></p>
</td>
<td class=cellinside>SMARCAD1
</td>
<td class=cellinside>SWI/SNF-related, matrix-associated actin-dependent regulator of chromatin, subfamily a, containing DEAD/H box 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6830">6830</a></p>
</td>
<td class=cellinside>SUPT6H
</td>
<td class=cellinside>SPT6 homolog, histone chaperone
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=7019">7019</a></p>
</td>
<td class=cellinside>TFAM
</td>
<td class=cellinside>transcription factor A, mitochondrial
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9760">9760</a></p>
</td>
<td class=cellinside>TOX
</td>
<td class=cellinside>thymocyte selection associated high mobility group box
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84969">84969</a></p>
</td>
<td class=cellinside>TOX2
</td>
<td class=cellinside>TOX high mobility group box family member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=27324">27324</a></p>
</td>
<td class=cellinside>TOX3
</td>
<td class=cellinside>TOX high mobility group box family member 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8805">8805</a></p>
</td>
<td class=cellinside>TRIM24
</td>
<td class=cellinside>tripartite motif containing 24
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84083">84083</a></p>
</td>
<td class=cellinside>ZRANB3
</td>
<td class=cellinside>zinc finger RANBP2-type containing 3
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>1.38e-01                                                                                                                                                                    </td>
<td class=cellinside>7.40e-09                                                                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043596">nuclear replication fork</a></p>                                                                 </td>
<td class=cellinside>cellular_component                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>1.09e-01                                                                                                                                                                    </td>
<td class=cellinside>1.89e-21                                                                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004386">helicase activity</a></p>                                                                        </td>
<td class=cellinside>molecular_function                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>9.62e-02                                                                                                                                                                    </td>
<td class=cellinside>2.71e-27                                                                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006338">chromatin remodeling</a></p>                                                                     </td>
<td class=cellinside>biological_process                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>7.02e-02                                                                                                                                                                    </td>
<td class=cellinside>6.79e-08                                                                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0070577">lysine-acetylated histone binding</a></p>                                                        </td>
<td class=cellinside>molecular_function                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>3.14e-02                                                                                                                                                                    </td>
<td class=cellinside>2.23e-04                                                                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0033234">negative regulation of protein sumoylation</a></p>                                               </td>
<td class=cellinside>biological_process                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>2.84e-02                                                                                                                                                                    </td>
<td class=cellinside>7.20e-06                                                                                                                                                                    </td>
<td class=cellinside>GO                                                                                                                                                                          </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008301">DNA binding, bending</a></p>                                                                     </td>
<td class=cellinside>molecular_function                                                                                                                                                          </td> </tr>
 <tr> 
<td class=cellinside>2.00e-02                                                                                                                                                                    </td>
<td class=cellinside>6.80e-03                                                                                                                                                                    </td>
<td class=cellinside>Reactome                                                                                                                                                                    </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-163282&species=Homo+sapiens&species=Entries+without+species&cluster=true">Mitochondrial transcription initiation</a></p></td>
<td class=cellinside>Gene expression (Transcription)                                                                                                                                             </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod298_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006259">DNA metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016569">covalent chromatin modification</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006325">chromatin organization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006338">chromatin remodeling</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032392">DNA geometric change</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071103">DNA conformation change</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0042393">histone binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004386">helicase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.07e-10
</td>
<td class=cellinside>1.22e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0017111">nucleoside-triphosphatase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.59e-10
</td>
<td class=cellinside>1.26e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016589">NURF complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.92e-10
</td>
<td class=cellinside>1.41e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0070603">SWI/SNF superfamily-type complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.01e-08
</td>
<td class=cellinside>6.57e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006974">cellular response to DNA damage stimulus</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.57e-08
</td>
<td class=cellinside>6.97e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0070577">lysine-acetylated histone binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.20e-08
</td>
<td class=cellinside>8.33e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000785">chromatin</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.25e-08
</td>
<td class=cellinside>1.40e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006281">DNA repair</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.68e-08
</td>
<td class=cellinside>1.65e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0034728">nucleosome organization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.09e-08
</td>
<td class=cellinside>1.67e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0003682">chromatin binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.36e-08
</td>
<td class=cellinside>2.89e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008094">DNA-dependent ATPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.47e-07
</td>
<td class=cellinside>7.89e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000792">heterochromatin</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.69e-07
</td>
<td class=cellinside>4.19e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006333">chromatin assembly or disassembly</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>3.30e-03
</td>
<td class=cellinside>3.25e-01
</td>
<td class=cellinside>SIGNALING BY FGFR1 MUTANTS
</td></tr>
 
<tr> 
<td class=cellinside>3.30e-03
</td>
<td class=cellinside>3.25e-01
</td>
<td class=cellinside>SIGNALING BY FGFR1 FUSION MUTANTS
</td></tr>
 
<tr> 
<td class=cellinside>3.31e-03
</td>
<td class=cellinside>3.26e-01
</td>
<td class=cellinside>SIGNALING BY FGFR MUTANTS
</td></tr>
 
<tr> 
<td class=cellinside>5.51e-03
</td>
<td class=cellinside>4.95e-01
</td>
<td class=cellinside>SIGNALING BY FGFR IN DISEASE
</td></tr>
 
<tr> 
<td class=cellinside>7.51e-03
</td>
<td class=cellinside>5.80e-01
</td>
<td class=cellinside>APOPTOSIS INDUCED DNA FRAGMENTATION
</td></tr>
 
<tr> 
<td class=cellinside>9.66e-03
</td>
<td class=cellinside>7.14e-01
</td>
<td class=cellinside>SIGNALING BY FGFR1 FUSION MUTANTS
</td></tr>
 
<tr> 
<td class=cellinside>9.68e-03
</td>
<td class=cellinside>7.14e-01
</td>
<td class=cellinside>SIGNALING BY FGFR1 MUTANTS
</td></tr>
 
<tr> 
<td class=cellinside>9.71e-03
</td>
<td class=cellinside>7.15e-01
</td>
<td class=cellinside>SIGNALING BY FGFR MUTANTS
</td></tr>
 
<tr> 
<td class=cellinside>1.29e-02
</td>
<td class=cellinside>9.73e-01
</td>
<td class=cellinside>DEPOSITION OF NEW CENPA CONTAINING NUCLEOSOMES AT THE CENTROMERE
</td></tr>
 
<tr> 
<td class=cellinside>1.39e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>APOPTOSIS INDUCED DNA FRAGMENTATION
</td></tr>
 
<tr> 
<td class=cellinside>1.50e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNALING BY FGFR IN DISEASE
</td></tr>
 
<tr> 
<td class=cellinside>1.78e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>DEPOSITION OF NEW CENPA CONTAINING NUCLEOSOMES AT THE CENTROMERE
</td></tr>
 
<tr> 
<td class=cellinside>2.28e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>CHROMOSOME MAINTENANCE
</td></tr>
 
<tr> 
<td class=cellinside>2.68e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>APOPTOTIC EXECUTION PHASE
</td></tr>
 
<tr> 
<td class=cellinside>2.73e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RNA POL I RNA POL III AND MITOCHONDRIAL TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>2.76e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>3.03e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>APOPTOTIC EXECUTION PHASE
</td></tr>
 
<tr> 
<td class=cellinside>3.38e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>CHROMOSOME MAINTENANCE
</td></tr>
 
<tr> 
<td class=cellinside>3.89e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RNA POL I RNA POL III AND MITOCHONDRIAL TRANSCRIPTION
</td></tr>
 
<tr> 
<td class=cellinside>3.95e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TRANSCRIPTION
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>4.04e-05
</td>
<td class=cellinside>1.24e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003387">aorta coarctation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.78e-04
</td>
<td class=cellinside>6.61e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004948">abnormal neuronal precursor proliferation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.37e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0006042">increased apoptosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.07e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005578">teratozoospermia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.12e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009703">decreased birth body size</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.30e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008973">decreased erythroid progenitor cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.18e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004200">decreased fetal size</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.20e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008963">increased carbon dioxide production</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.38e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008397">abnormal CD4-positive, CD25-positive, alpha-beta regulatory T cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.01e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001730">embryonic growth arrest</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.05e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0006362">abnormal male germ cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.06e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000934">abnormal telencephalon development</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004290">abnormal stapes footplate morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002859">abnormal inner ear canal fusion</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010220">decreased T-helper 17 cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.13e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004718">abnormal vestibular nerve morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.13e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010526">aortic arch coarctation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012167">abnormal epigenetic regulation of gene expression</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005635">decreased circulating bilirubin level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0013466">keratoconjunctivitis sicca</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 23 09:59:09 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>