<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod186</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod186</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>21 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84226">84226</a></p>
</td>
<td class=cellinside>C2orf16
</td>
<td class=cellinside>chromosome 2 open reading frame 16
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1400">1400</a></p>
</td>
<td class=cellinside>CRMP1
</td>
<td class=cellinside>collapsin response mediator protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51706">51706</a></p>
</td>
<td class=cellinside>CYB5R1
</td>
<td class=cellinside>cytochrome b5 reductase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51700">51700</a></p>
</td>
<td class=cellinside>CYB5R2
</td>
<td class=cellinside>cytochrome b5 reductase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1727">1727</a></p>
</td>
<td class=cellinside>CYB5R3
</td>
<td class=cellinside>cytochrome b5 reductase 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51167">51167</a></p>
</td>
<td class=cellinside>CYB5R4
</td>
<td class=cellinside>cytochrome b5 reductase 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=606495">606495</a></p>
</td>
<td class=cellinside>CYB5RL
</td>
<td class=cellinside>cytochrome b5 reductase like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1723">1723</a></p>
</td>
<td class=cellinside>DHODH
</td>
<td class=cellinside>dihydroorotate dehydrogenase (quinone)
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1806">1806</a></p>
</td>
<td class=cellinside>DPYD
</td>
<td class=cellinside>dihydropyrimidine dehydrogenase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1807">1807</a></p>
</td>
<td class=cellinside>DPYS
</td>
<td class=cellinside>dihydropyrimidinase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1808">1808</a></p>
</td>
<td class=cellinside>DPYSL2
</td>
<td class=cellinside>dihydropyrimidinase like 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1809">1809</a></p>
</td>
<td class=cellinside>DPYSL3
</td>
<td class=cellinside>dihydropyrimidinase like 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10570">10570</a></p>
</td>
<td class=cellinside>DPYSL4
</td>
<td class=cellinside>dihydropyrimidinase like 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56896">56896</a></p>
</td>
<td class=cellinside>DPYSL5
</td>
<td class=cellinside>dihydropyrimidinase like 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2232">2232</a></p>
</td>
<td class=cellinside>FDXR
</td>
<td class=cellinside>ferredoxin reductase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=83943">83943</a></p>
</td>
<td class=cellinside>IMMP2L
</td>
<td class=cellinside>inner mitochondrial membrane peptidase subunit 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=92106">92106</a></p>
</td>
<td class=cellinside>OXNAD1
</td>
<td class=cellinside>oxidoreductase NAD binding domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9963">9963</a></p>
</td>
<td class=cellinside>SLC23A1
</td>
<td class=cellinside>solute carrier family 23 member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9962">9962</a></p>
</td>
<td class=cellinside>SLC23A2
</td>
<td class=cellinside>solute carrier family 23 member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=151295">151295</a></p>
</td>
<td class=cellinside>SLC23A3
</td>
<td class=cellinside>solute carrier family 23 member 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=7372">7372</a></p>
</td>
<td class=cellinside>UMPS
</td>
<td class=cellinside>uridine monophosphate synthetase
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>3.26e-01                                                                                                                                                                                     </td>
<td class=cellinside>3.48e-06                                                                                                                                                                                     </td>
<td class=cellinside>Reactome                                                                                                                                                                                     </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-500753&species=Homo+sapiens&species=Entries+without+species&cluster=true">Pyrimidine biosynthesis</a></p>                                </td>
<td class=cellinside>Metabolism                                                                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>2.76e-01                                                                                                                                                                                     </td>
<td class=cellinside>1.16e-06                                                                                                                                                                                     </td>
<td class=cellinside>GO                                                                                                                                                                                           </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0015229">L-ascorbic acid transmembrane transporter activity</a></p>                                                        </td>
<td class=cellinside>molecular_function                                                                                                                                                                           </td> </tr>
 <tr> 
<td class=cellinside>2.75e-01                                                                                                                                                                                     </td>
<td class=cellinside>1.16e-06                                                                                                                                                                                     </td>
<td class=cellinside>GO                                                                                                                                                                                           </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0015882">L-ascorbic acid transmembrane transport</a></p>                                                                   </td>
<td class=cellinside>biological_process                                                                                                                                                                           </td> </tr>
 <tr> 
<td class=cellinside>1.55e-01                                                                                                                                                                                     </td>
<td class=cellinside>4.05e-12                                                                                                                                                                                     </td>
<td class=cellinside>Reactome                                                                                                                                                                                     </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-399956&species=Homo+sapiens&species=Entries+without+species&cluster=true">CRMPs in Sema3A signaling</a></p>                              </td>
<td class=cellinside>Developmental Biology                                                                                                                                                                        </td> </tr>
 <tr> 
<td class=cellinside>8.63e-02                                                                                                                                                                                     </td>
<td class=cellinside>5.26e-10                                                                                                                                                                                     </td>
<td class=cellinside>Reactome                                                                                                                                                                                     </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-1237044&species=Homo+sapiens&species=Entries+without+species&cluster=true">Erythrocytes take up carbon dioxide and release oxygen</a></p></td>
<td class=cellinside>Transport of small molecules                                                                                                                                                                 </td> </tr>
 <tr> 
<td class=cellinside>8.57e-02                                                                                                                                                                                     </td>
<td class=cellinside>5.26e-10                                                                                                                                                                                     </td>
<td class=cellinside>Reactome                                                                                                                                                                                     </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-1480926&species=Homo+sapiens&species=Entries+without+species&cluster=true">O2/CO2 exchange in erythrocytes</a></p>                       </td>
<td class=cellinside>Transport of small molecules                                                                                                                                                                 </td> </tr>
 <tr> 
<td class=cellinside>4.67e-02                                                                                                                                                                                     </td>
<td class=cellinside>6.40e-08                                                                                                                                                                                     </td>
<td class=cellinside>Reactome                                                                                                                                                                                     </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-196836&species=Homo+sapiens&species=Entries+without+species&cluster=true">Vitamin C (ascorbate) metabolism</a></p>                       </td>
<td class=cellinside>Metabolism                                                                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>4.45e-02                                                                                                                                                                                     </td>
<td class=cellinside>3.26e-07                                                                                                                                                                                     </td>
<td class=cellinside>GO                                                                                                                                                                                           </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031005">filamin binding</a></p>                                                                                           </td>
<td class=cellinside>molecular_function                                                                                                                                                                           </td> </tr>
 <tr> 
<td class=cellinside>2.83e-02                                                                                                                                                                                     </td>
<td class=cellinside>3.32e-03                                                                                                                                                                                     </td>
<td class=cellinside>Reactome                                                                                                                                                                                     </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-2395516&species=Homo+sapiens&species=Entries+without+species&cluster=true">Electron transport from NADPH to Ferredoxin</a></p>           </td>
<td class=cellinside>Metabolism                                                                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>2.15e-02                                                                                                                                                                                     </td>
<td class=cellinside>6.34e-05                                                                                                                                                                                     </td>
<td class=cellinside>GO                                                                                                                                                                                           </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0046135">pyrimidine nucleoside catabolic process</a></p>                                                                   </td>
<td class=cellinside>biological_process                                                                                                                                                                           </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod186_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>8.10e-13
</td>
<td class=cellinside>5.61e-10
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004128">cytochrome-b5 reductase activity, acting on NAD(P)H</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.45e-12
</td>
<td class=cellinside>1.68e-09
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016653">oxidoreductase activity, acting on NAD(P)H, heme protein as acceptor</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.94e-08
</td>
<td class=cellinside>1.22e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006206">pyrimidine nucleobase metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.16e-08
</td>
<td class=cellinside>1.32e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016810">hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.19e-07
</td>
<td class=cellinside>4.51e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031005">filamin binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.83e-07
</td>
<td class=cellinside>6.71e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016651">oxidoreductase activity, acting on NAD(P)H</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.23e-07
</td>
<td class=cellinside>8.05e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016491">oxidoreductase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.68e-07
</td>
<td class=cellinside>2.36e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0015701">bicarbonate transport</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.67e-06
</td>
<td class=cellinside>7.61e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0019852">L-ascorbic acid metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.99e-06
</td>
<td class=cellinside>2.06e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006213">pyrimidine nucleoside metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.52e-05
</td>
<td class=cellinside>5.63e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0015711">organic anion transport</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.91e-05
</td>
<td class=cellinside>6.90e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0019860">uracil metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.91e-05
</td>
<td class=cellinside>6.90e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0044205">'de novo' UMP biosynthetic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.39e-05
</td>
<td class=cellinside>8.40e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006207">'de novo' pyrimidine nucleobase biosynthetic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.85e-05
</td>
<td class=cellinside>6.81e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016812">hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.90e-05
</td>
<td class=cellinside>9.97e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016125">sterol metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.98e-05
</td>
<td class=cellinside>1.02e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0019859">thymine metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.98e-05
</td>
<td class=cellinside>1.02e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006210">thymine catabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.98e-05
</td>
<td class=cellinside>1.02e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006208">pyrimidine nucleobase catabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.39e-05
</td>
<td class=cellinside>1.14e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006694">steroid biosynthetic process</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>3.06e-11
</td>
<td class=cellinside>1.15e-08
</td>
<td class=cellinside>CRMPS IN SEMA3A SIGNALING
</td></tr>
 
<tr> 
<td class=cellinside>1.06e-10
</td>
<td class=cellinside>2.89e-08
</td>
<td class=cellinside>CRMPS IN SEMA3A SIGNALING
</td></tr>
 
<tr> 
<td class=cellinside>8.30e-09
</td>
<td class=cellinside>2.46e-06
</td>
<td class=cellinside>SEMAPHORIN INTERACTIONS
</td></tr>
 
<tr> 
<td class=cellinside>7.37e-08
</td>
<td class=cellinside>1.46e-05
</td>
<td class=cellinside>SEMAPHORIN INTERACTIONS
</td></tr>
 
<tr> 
<td class=cellinside>2.41e-07
</td>
<td class=cellinside>5.94e-05
</td>
<td class=cellinside>PYRIMIDINE METABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>3.33e-07
</td>
<td class=cellinside>6.15e-05
</td>
<td class=cellinside>PYRIMIDINE METABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>9.21e-06
</td>
<td class=cellinside>1.77e-03
</td>
<td class=cellinside>AXON GUIDANCE
</td></tr>
 
<tr> 
<td class=cellinside>2.87e-05
</td>
<td class=cellinside>5.05e-03
</td>
<td class=cellinside>METABOLISM OF NUCLEOTIDES
</td></tr>
 
<tr> 
<td class=cellinside>4.18e-05
</td>
<td class=cellinside>5.68e-03
</td>
<td class=cellinside>AXON GUIDANCE
</td></tr>
 
<tr> 
<td class=cellinside>5.41e-05
</td>
<td class=cellinside>7.24e-03
</td>
<td class=cellinside>METABOLISM OF NUCLEOTIDES
</td></tr>
 
<tr> 
<td class=cellinside>1.05e-04
</td>
<td class=cellinside>1.62e-02
</td>
<td class=cellinside>METABOLISM OF VITAMINS AND COFACTORS
</td></tr>
 
<tr> 
<td class=cellinside>1.34e-04
</td>
<td class=cellinside>2.02e-02
</td>
<td class=cellinside>DEVELOPMENTAL BIOLOGY
</td></tr>
 
<tr> 
<td class=cellinside>3.15e-04
</td>
<td class=cellinside>4.29e-02
</td>
<td class=cellinside>PYRIMIDINE CATABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>6.71e-04
</td>
<td class=cellinside>7.04e-02
</td>
<td class=cellinside>METABOLISM OF VITAMINS AND COFACTORS
</td></tr>
 
<tr> 
<td class=cellinside>8.49e-04
</td>
<td class=cellinside>8.68e-02
</td>
<td class=cellinside>PYRIMIDINE CATABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>1.85e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RECYCLING PATHWAY OF L1
</td></tr>
 
<tr> 
<td class=cellinside>2.21e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RECYCLING PATHWAY OF L1
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.96e-05
</td>
<td class=cellinside>7.47e-02
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011229">abnormal vitamin C level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.30e-04
</td>
<td class=cellinside>4.74e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009940">abnormal hippocampus pyramidal cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.87e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008291">abnormal adrenocortical cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.87e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002164">abnormal gland physiology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.32e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001905">abnormal dopamine level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001379">abnormal penile erection</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001671">abnormal vitamin absorption</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001548">hyperlipidemia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005556">abnormal renal filtration rate</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.36e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001900">impaired synaptic plasticity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.36e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002665">decreased circulating corticosterone level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.09e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008572">abnormal Purkinje cell dendrite morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001899">absent long term depression</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003339">decreased pancreatic beta cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001739">abnormal adrenal gland secretion</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.30e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003962">abnormal adrenaline level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008263">abnormal hippocampus CA1 region morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.68e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008284">abnormal hippocampus pyramidal cell layer</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.23e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001132">absent mature ovarian follicles</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.41e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003311">aminoaciduria</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 23 09:53:39 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>