<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
<p class='character'><a href="../index.html">Back to main page</a></p>

 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod173</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod173</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>30 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51412">51412</a></p>
</td>
<td class=cellinside>ACTL6B
</td>
<td class=cellinside>actin like 6B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8289">8289</a></p>
</td>
<td class=cellinside>ARID1A
</td>
<td class=cellinside>AT-rich interaction domain 1A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57492">57492</a></p>
</td>
<td class=cellinside>ARID1B
</td>
<td class=cellinside>AT-rich interaction domain 1B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9274">9274</a></p>
</td>
<td class=cellinside>BCL7C
</td>
<td class=cellinside>BCL tumor suppressor 7C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=29117">29117</a></p>
</td>
<td class=cellinside>BRD7
</td>
<td class=cellinside>bromodomain containing 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=119710">119710</a></p>
</td>
<td class=cellinside>C11orf74
</td>
<td class=cellinside>chromosome 11 open reading frame 74
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=78996">78996</a></p>
</td>
<td class=cellinside>C7orf49
</td>
<td class=cellinside>cell cycle regulator of NHEJ
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8193">8193</a></p>
</td>
<td class=cellinside>DPF1
</td>
<td class=cellinside>double PHD fingers 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5977">5977</a></p>
</td>
<td class=cellinside>DPF2
</td>
<td class=cellinside>double PHD fingers 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8110">8110</a></p>
</td>
<td class=cellinside>DPF3
</td>
<td class=cellinside>double PHD fingers 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=3070">3070</a></p>
</td>
<td class=cellinside>HELLS
</td>
<td class=cellinside>helicase, lymphoid specific
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=3146">3146</a></p>
</td>
<td class=cellinside>HMGB1
</td>
<td class=cellinside>high mobility group box 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51603">51603</a></p>
</td>
<td class=cellinside>METTL13
</td>
<td class=cellinside>methyltransferase like 13
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55193">55193</a></p>
</td>
<td class=cellinside>PBRM1
</td>
<td class=cellinside>polybromo 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55274">55274</a></p>
</td>
<td class=cellinside>PHF10
</td>
<td class=cellinside>PHD finger protein 10
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5989">5989</a></p>
</td>
<td class=cellinside>RFX1
</td>
<td class=cellinside>regulatory factor X1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5990">5990</a></p>
</td>
<td class=cellinside>RFX2
</td>
<td class=cellinside>regulatory factor X2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5991">5991</a></p>
</td>
<td class=cellinside>RFX3
</td>
<td class=cellinside>regulatory factor X3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5992">5992</a></p>
</td>
<td class=cellinside>RFX4
</td>
<td class=cellinside>regulatory factor X4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5993">5993</a></p>
</td>
<td class=cellinside>RFX5
</td>
<td class=cellinside>regulatory factor X5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=222546">222546</a></p>
</td>
<td class=cellinside>RFX6
</td>
<td class=cellinside>regulatory factor X6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=731220">731220</a></p>
</td>
<td class=cellinside>RFX8
</td>
<td class=cellinside>RFX family member 8, lacking RFX DNA binding domain
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6595">6595</a></p>
</td>
<td class=cellinside>SMARCA2
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily a, member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6598">6598</a></p>
</td>
<td class=cellinside>SMARCB1
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily b, member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6599">6599</a></p>
</td>
<td class=cellinside>SMARCC1
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin subfamily c member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6601">6601</a></p>
</td>
<td class=cellinside>SMARCC2
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin subfamily c member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6602">6602</a></p>
</td>
<td class=cellinside>SMARCD1
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily d, member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6603">6603</a></p>
</td>
<td class=cellinside>SMARCD2
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily d, member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6605">6605</a></p>
</td>
<td class=cellinside>SMARCE1
</td>
<td class=cellinside>SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily e, member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=126637">126637</a></p>
</td>
<td class=cellinside>TCHHL1
</td>
<td class=cellinside>trichohyalin like 1
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>4.96e-01                                                                                                                                                                                                                 </td>
<td class=cellinside>3.20e-33                                                                                                                                                                                                                 </td>
<td class=cellinside>GO                                                                                                                                                                                                                       </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071565">nBAF complex</a></p>                                                                                                                          </td>
<td class=cellinside>cellular_component                                                                                                                                                                                                       </td> </tr>
 <tr> 
<td class=cellinside>1.24e-01                                                                                                                                                                                                                 </td>
<td class=cellinside>2.59e-26                                                                                                                                                                                                                 </td>
<td class=cellinside>GO                                                                                                                                                                                                                       </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016514">SWI/SNF complex</a></p>                                                                                                                       </td>
<td class=cellinside>cellular_component                                                                                                                                                                                                       </td> </tr>
 <tr> 
<td class=cellinside>1.89e-02                                                                                                                                                                                                                 </td>
<td class=cellinside>4.11e-21                                                                                                                                                                                                                 </td>
<td class=cellinside>GO                                                                                                                                                                                                                       </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071564">npBAF complex</a></p>                                                                                                                         </td>
<td class=cellinside>cellular_component                                                                                                                                                                                                       </td> </tr>
 <tr> 
<td class=cellinside>1.85e-02                                                                                                                                                                                                                 </td>
<td class=cellinside>6.23e-19                                                                                                                                                                                                                 </td>
<td class=cellinside>GO                                                                                                                                                                                                                       </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006337">nucleosome disassembly</a></p>                                                                                                                </td>
<td class=cellinside>biological_process                                                                                                                                                                                                       </td> </tr>
 <tr> 
<td class=cellinside>8.42e-03                                                                                                                                                                                                                 </td>
<td class=cellinside>6.75e-24                                                                                                                                                                                                                 </td>
<td class=cellinside>Reactome                                                                                                                                                                                                                 </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-8939243&species=Homo+sapiens&species=Entries+without+species&cluster=true">RUNX1 interacts with co-factors whose precise effect on RUNX1 targets is not known</a></p></td>
<td class=cellinside>Gene expression (Transcription)                                                                                                                                                                                          </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod173_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016569">covalent chromatin modification</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006338">chromatin remodeling</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006325">chromatin organization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006337">nucleosome disassembly</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016514">SWI/SNF complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0090544">BAF-type complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071564">npBAF complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071565">nBAF complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0070603">SWI/SNF superfamily-type complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.63e-11
</td>
<td class=cellinside>2.30e-08
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006333">chromatin assembly or disassembly</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.17e-10
</td>
<td class=cellinside>1.74e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043044">ATP-dependent chromatin remodeling</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.14e-09
</td>
<td class=cellinside>8.43e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0034728">nucleosome organization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.23e-09
</td>
<td class=cellinside>6.43e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000978">RNA polymerase II core promoter proximal region sequence-specific DNA binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.88e-09
</td>
<td class=cellinside>9.56e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000987">core promoter proximal region sequence-specific DNA binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.06e-09
</td>
<td class=cellinside>1.04e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0001159">core promoter proximal region DNA binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.76e-09
</td>
<td class=cellinside>3.50e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000790">nuclear chromatin</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.38e-08
</td>
<td class=cellinside>5.37e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000785">chromatin</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.32e-08
</td>
<td class=cellinside>1.58e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000228">nuclear chromosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.21e-08
</td>
<td class=cellinside>2.09e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0044212">transcription regulatory region DNA binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.24e-08
</td>
<td class=cellinside>2.10e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000975">regulatory region DNA binding</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>8.78e-03
</td>
<td class=cellinside>7.21e-01
</td>
<td class=cellinside>APOPTOSIS INDUCED DNA FRAGMENTATION
</td></tr>
 
<tr> 
<td class=cellinside>1.07e-02
</td>
<td class=cellinside>8.40e-01
</td>
<td class=cellinside>ADVANCED GLYCOSYLATION ENDPRODUCT RECEPTOR SIGNALING
</td></tr>
 
<tr> 
<td class=cellinside>1.88e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RIP MEDIATED NFKB ACTIVATION VIA DAI
</td></tr>
 
<tr> 
<td class=cellinside>2.21e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TRAF6 MEDIATED NFKB ACTIVATION
</td></tr>
 
<tr> 
<td class=cellinside>2.88e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TAK1 ACTIVATES NFKB BY PHOSPHORYLATION AND ACTIVATION OF IKKS COMPLEX
</td></tr>
 
<tr> 
<td class=cellinside>3.23e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>APOPTOTIC EXECUTION PHASE
</td></tr>
 
<tr> 
<td class=cellinside>4.54e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>RIG I MDA5 MEDIATED INDUCTION OF IFN ALPHA BETA PATHWAYS
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>7.62e-04
</td>
<td class=cellinside>7.68e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000787">abnormal telencephalon morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.74e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0013598">Leydig cell hypertrophy</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.75e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004023">abnormal chromosome number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004153">increased renal tubule apoptosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0013207">absent endoderm</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.98e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010402">ventricular septal defect</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.33e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000927">small floor plate</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.33e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002958">cerebral aqueductal stenosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.33e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012118">absent trophectoderm cell proliferation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009277">increased brain tumor incidence</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000293">absent myocardial trabeculae</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.35e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003270">intestinal obstruction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000650">mesocardia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009429">decreased embryo weight</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012119">increased trophectoderm apoptosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004269">abnormal optic cup morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002399">abnormal pluripotent precursor cell morphology/development</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004154">renal tubular necrosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.49e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000823">abnormal lateral ventricle morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.67e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004838">abnormal neural fold elevation formation</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 30 17:28:19 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>