<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
<p class='character'><a href="../index.html">Back to main page</a></p>

 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod359</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod359</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>44 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1244">1244</a></p>
</td>
<td class=cellinside>ABCC2
</td>
<td class=cellinside>ATP binding cassette subfamily C member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=123624">123624</a></p>
</td>
<td class=cellinside>AGBL1
</td>
<td class=cellinside>ATP/GTP binding protein like 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23287">23287</a></p>
</td>
<td class=cellinside>AGTPBP1
</td>
<td class=cellinside>ATP/GTP binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10550">10550</a></p>
</td>
<td class=cellinside>ARL6IP5
</td>
<td class=cellinside>ADP ribosylation factor like GTPase 6 interacting protein 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=100910172">100910172</a></p>
</td>
<td class=cellinside>BASP1
</td>
<td class=cellinside>brain acid soluble protein 1-like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=790955">790955</a></p>
</td>
<td class=cellinside>C11orf83
</td>
<td class=cellinside>ubiquinol-cytochrome c reductase complex assembly factor 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=125988">125988</a></p>
</td>
<td class=cellinside>C19orf70
</td>
<td class=cellinside>chromosome 19 open reading frame 70
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79095">79095</a></p>
</td>
<td class=cellinside>C9orf16
</td>
<td class=cellinside>chromosome 9 open reading frame 16
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=158055">158055</a></p>
</td>
<td class=cellinside>C9orf163
</td>
<td class=cellinside>chromosome 9 open reading frame 163
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1604">1604</a></p>
</td>
<td class=cellinside>CD55
</td>
<td class=cellinside>CD55 molecule (Cromer blood group)
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55573">55573</a></p>
</td>
<td class=cellinside>CDV3
</td>
<td class=cellinside>CDV3 homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1203">1203</a></p>
</td>
<td class=cellinside>CLN5
</td>
<td class=cellinside>CLN5, intracellular trafficking protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=58190">58190</a></p>
</td>
<td class=cellinside>CTDSP1
</td>
<td class=cellinside>CTD small phosphatase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10106">10106</a></p>
</td>
<td class=cellinside>CTDSP2
</td>
<td class=cellinside>CTD small phosphatase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10217">10217</a></p>
</td>
<td class=cellinside>CTDSPL
</td>
<td class=cellinside>CTD small phosphatase like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23549">23549</a></p>
</td>
<td class=cellinside>DAP
</td>
<td class=cellinside>aspartyl aminopeptidase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10613">10613</a></p>
</td>
<td class=cellinside>ERLIN1
</td>
<td class=cellinside>ER lipid raft associated 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55177">55177</a></p>
</td>
<td class=cellinside>FAM82A2
</td>
<td class=cellinside>regulator of microtubule dynamics 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2319">2319</a></p>
</td>
<td class=cellinside>FLOT2
</td>
<td class=cellinside>flotillin 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2631">2631</a></p>
</td>
<td class=cellinside>GBAS
</td>
<td class=cellinside>nipsnap homolog 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51155">51155</a></p>
</td>
<td class=cellinside>HN1
</td>
<td class=cellinside>Jupiter microtubule associated homolog 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=121457">121457</a></p>
</td>
<td class=cellinside>IKBIP
</td>
<td class=cellinside>IKBKB interacting protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55004">55004</a></p>
</td>
<td class=cellinside>LAMTOR1
</td>
<td class=cellinside>late endosomal/lysosomal adaptor, MAPK and MTOR activator 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=28956">28956</a></p>
</td>
<td class=cellinside>LAMTOR2
</td>
<td class=cellinside>late endosomal/lysosomal adaptor, MAPK and MTOR activator 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=389541">389541</a></p>
</td>
<td class=cellinside>LAMTOR4
</td>
<td class=cellinside>late endosomal/lysosomal adaptor, MAPK and MTOR activator 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=137994">137994</a></p>
</td>
<td class=cellinside>LETM2
</td>
<td class=cellinside>leucine zipper and EF-hand containing transmembrane protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64757">64757</a></p>
</td>
<td class=cellinside>MARC1
</td>
<td class=cellinside>mitochondrial amidoxime reducing component 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57192">57192</a></p>
</td>
<td class=cellinside>MCOLN1
</td>
<td class=cellinside>mucolipin 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8508">8508</a></p>
</td>
<td class=cellinside>NIPSNAP1
</td>
<td class=cellinside>nipsnap homolog 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5445">5445</a></p>
</td>
<td class=cellinside>PON2
</td>
<td class=cellinside>paraoxonase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11230">11230</a></p>
</td>
<td class=cellinside>PRAF2
</td>
<td class=cellinside>PRA1 domain family member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51651">51651</a></p>
</td>
<td class=cellinside>PTRH2
</td>
<td class=cellinside>peptidyl-tRNA hydrolase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=950">950</a></p>
</td>
<td class=cellinside>SCARB2
</td>
<td class=cellinside>scavenger receptor class B member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=29106">29106</a></p>
</td>
<td class=cellinside>SCG3
</td>
<td class=cellinside>secretogranin III
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23677">23677</a></p>
</td>
<td class=cellinside>SH3BP4
</td>
<td class=cellinside>SH3 domain binding protein 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=91137">91137</a></p>
</td>
<td class=cellinside>SLC25A46
</td>
<td class=cellinside>solute carrier family 25 member 46
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=125170">125170</a></p>
</td>
<td class=cellinside>SMCR7
</td>
<td class=cellinside>mitochondrial elongation factor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55240">55240</a></p>
</td>
<td class=cellinside>STEAP3
</td>
<td class=cellinside>STEAP3 metalloreductase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55333">55333</a></p>
</td>
<td class=cellinside>SYNJ2BP
</td>
<td class=cellinside>synaptojanin 2 binding protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10548">10548</a></p>
</td>
<td class=cellinside>TM9SF1
</td>
<td class=cellinside>transmembrane 9 superfamily member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84233">84233</a></p>
</td>
<td class=cellinside>TMEM126A
</td>
<td class=cellinside>transmembrane protein 126A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=81542">81542</a></p>
</td>
<td class=cellinside>TMX1
</td>
<td class=cellinside>thioredoxin related transmembrane protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84817">84817</a></p>
</td>
<td class=cellinside>TXNDC17
</td>
<td class=cellinside>thioredoxin domain containing 17
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80139">80139</a></p>
</td>
<td class=cellinside>ZNF703
</td>
<td class=cellinside>zinc finger protein 703
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>2.11e-01                                                                                                                                                        </td>
<td class=cellinside>2.25e-07                                                                                                                                                        </td>
<td class=cellinside>GO                                                                                                                                                              </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008420">CTD phosphatase activity</a></p>                                                     </td>
<td class=cellinside>molecular_function                                                                                                                                              </td> </tr>
 <tr> 
<td class=cellinside>2.51e-02                                                                                                                                                        </td>
<td class=cellinside>8.39e-06                                                                                                                                                        </td>
<td class=cellinside>Reactome                                                                                                                                                        </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-166208&species=Homo+sapiens&species=Entries+without+species&cluster=true">mTORC1-mediated signalling</a></p></td>
<td class=cellinside>Signal Transduction                                                                                                                                             </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod359_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.69e-06
</td>
<td class=cellinside>4.76e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071986">Ragulator complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.19e-05
</td>
<td class=cellinside>4.53e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006914">autophagy</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.45e-05
</td>
<td class=cellinside>5.39e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071230">cellular response to amino acid stimulus</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.59e-05
</td>
<td class=cellinside>9.02e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0061462">protein localization to lysosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.26e-05
</td>
<td class=cellinside>1.10e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032006">regulation of TOR signaling</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.46e-05
</td>
<td class=cellinside>1.16e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035609">C-terminal protein deglutamylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.68e-05
</td>
<td class=cellinside>8.03e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031902">late endosome membrane</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.87e-05
</td>
<td class=cellinside>1.28e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032008">positive regulation of TOR signaling</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.90e-05
</td>
<td class=cellinside>1.88e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008420">CTD phosphatase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.03e-04
</td>
<td class=cellinside>2.01e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005770">late endosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.15e-04
</td>
<td class=cellinside>3.29e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043200">response to amino acid</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.77e-04
</td>
<td class=cellinside>4.74e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035610">protein side chain deglutamylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.94e-04
</td>
<td class=cellinside>3.51e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005765">lysosomal membrane</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.07e-04
</td>
<td class=cellinside>3.88e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032947">protein complex scaffold</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.26e-04
</td>
<td class=cellinside>5.81e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035608">protein deglutamylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.71e-04
</td>
<td class=cellinside>6.77e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016236">macroautophagy</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.60e-04
</td>
<td class=cellinside>6.07e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005774">vacuolar membrane</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.23e-04
</td>
<td class=cellinside>8.39e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0010008">endosome membrane</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.50e-04
</td>
<td class=cellinside>1.39e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A2000134">negative regulation of G1/S transition of mitotic cell cycle</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.51e-04
</td>
<td class=cellinside>1.39e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1902807">negative regulation of cell cycle G1/S phase transition</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>2.07e-05
</td>
<td class=cellinside>2.95e-03
</td>
<td class=cellinside>TRANSFERRIN ENDOCYTOSIS AND RECYCLING
</td></tr>
 
<tr> 
<td class=cellinside>3.54e-04
</td>
<td class=cellinside>3.98e-02
</td>
<td class=cellinside>IRON UPTAKE AND TRANSPORT
</td></tr>
 
<tr> 
<td class=cellinside>4.46e-04
</td>
<td class=cellinside>5.82e-02
</td>
<td class=cellinside>TRANSFERRIN ENDOCYTOSIS AND RECYCLING
</td></tr>
 
<tr> 
<td class=cellinside>1.11e-03
</td>
<td class=cellinside>1.29e-01
</td>
<td class=cellinside>IRON UPTAKE AND TRANSPORT
</td></tr>
 
<tr> 
<td class=cellinside>1.22e-02
</td>
<td class=cellinside>8.65e-01
</td>
<td class=cellinside>REGULATION OF COMPLEMENT CASCADE
</td></tr>
 
<tr> 
<td class=cellinside>1.46e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TRANSMEMBRANE TRANSPORT OF SMALL MOLECULES
</td></tr>
 
<tr> 
<td class=cellinside>1.75e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>REGULATION OF COMPLEMENT CASCADE
</td></tr>
 
<tr> 
<td class=cellinside>2.00e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>ACTIVATION OF CHAPERONE GENES BY XBP1S
</td></tr>
 
<tr> 
<td class=cellinside>2.85e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>UNFOLDED PROTEIN RESPONSE
</td></tr>
 
<tr> 
<td class=cellinside>3.08e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>COMPLEMENT CASCADE
</td></tr>
 
<tr> 
<td class=cellinside>4.27e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>COMPLEMENT CASCADE
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>7.51e-04
</td>
<td class=cellinside>7.64e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001106">abnormal Schwann cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.02e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004738">abnormal auditory brainstem response</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.20e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003731">abnormal retinal outer nuclear layer morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.00e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001270">distended abdomen</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.41e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005505">thrombocytosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.28e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003632">abnormal nervous system morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.65e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000480">increased rib number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0006349">decreased circulating copper level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010175">leptocytosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.47e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004438">abnormal vestibular hair cell physiology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009341">decreased splenocyte apoptosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002895">abnormal otolithic membrane morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.33e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003066">increased liver copper level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002424">abnormal reticulocyte morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012490">abnormal cochlear VIII nucleus morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004243">abnormal motor nerve collateral sprouting</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009954">abnormal mitral cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004366">abnormal strial marginal cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011426">abnormal ureter smooth muscle morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004410">absent endocochlear potential</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 30 17:37:24 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>