<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
<p class='character'><a href="../index.html">Back to main page</a></p>

 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod362</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod362</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>51 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8751">8751</a></p>
</td>
<td class=cellinside>ADAM15
</td>
<td class=cellinside>ADAM metallopeptidase domain 15
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84632">84632</a></p>
</td>
<td class=cellinside>AFAP1L2
</td>
<td class=cellinside>actin filament associated protein 1 like 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9472">9472</a></p>
</td>
<td class=cellinside>AKAP6
</td>
<td class=cellinside>A-kinase anchoring protein 6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10077">10077</a></p>
</td>
<td class=cellinside>ART1
</td>
<td class=cellinside>tetraspanin 32
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64921">64921</a></p>
</td>
<td class=cellinside>CASD1
</td>
<td class=cellinside>CAS1 domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64866">64866</a></p>
</td>
<td class=cellinside>CDCP1
</td>
<td class=cellinside>CUB domain containing protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6792">6792</a></p>
</td>
<td class=cellinside>CDKL5
</td>
<td class=cellinside>cyclin dependent kinase like 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1398">1398</a></p>
</td>
<td class=cellinside>CRK
</td>
<td class=cellinside>CRK proto-oncogene, adaptor protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=131566">131566</a></p>
</td>
<td class=cellinside>DCBLD2
</td>
<td class=cellinside>discoidin, CUB and LCCL domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=27120">27120</a></p>
</td>
<td class=cellinside>DKKL1
</td>
<td class=cellinside>dickkopf like acrosomal protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1794">1794</a></p>
</td>
<td class=cellinside>DOCK2
</td>
<td class=cellinside>dedicator of cytokinesis 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1795">1795</a></p>
</td>
<td class=cellinside>DOCK3
</td>
<td class=cellinside>dedicator of cytokinesis 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9732">9732</a></p>
</td>
<td class=cellinside>DOCK4
</td>
<td class=cellinside>dedicator of cytokinesis 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10278">10278</a></p>
</td>
<td class=cellinside>EFS
</td>
<td class=cellinside>embryonal Fyn-associated substrate
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79574">79574</a></p>
</td>
<td class=cellinside>EPS8L3
</td>
<td class=cellinside>EPS8 like 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54206">54206</a></p>
</td>
<td class=cellinside>ERRFI1
</td>
<td class=cellinside>ERBB receptor feedback inhibitor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=28964">28964</a></p>
</td>
<td class=cellinside>GIT1
</td>
<td class=cellinside>GIT ArfGAP 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=29998">29998</a></p>
</td>
<td class=cellinside>GLTSCR1
</td>
<td class=cellinside>BRD4 interacting chromatin remodeling complex associated protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11250">11250</a></p>
</td>
<td class=cellinside>GPR45
</td>
<td class=cellinside>G protein-coupled receptor 45
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=3082">3082</a></p>
</td>
<td class=cellinside>HGF
</td>
<td class=cellinside>hepatocyte growth factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84941">84941</a></p>
</td>
<td class=cellinside>HSH2D
</td>
<td class=cellinside>hematopoietic SH2 domain containing
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8692">8692</a></p>
</td>
<td class=cellinside>HYAL2
</td>
<td class=cellinside>hyaluronidase 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55243">55243</a></p>
</td>
<td class=cellinside>KIRREL
</td>
<td class=cellinside>kirre like nephrin family adhesion molecule 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=259215">259215</a></p>
</td>
<td class=cellinside>LY6G6F
</td>
<td class=cellinside>lymphocyte antigen 6 family member G6F
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57591">57591</a></p>
</td>
<td class=cellinside>MAL
</td>
<td class=cellinside>myocardin related transcription factor A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4301">4301</a></p>
</td>
<td class=cellinside>MLLT4
</td>
<td class=cellinside>afadin, adherens junction formation factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=344148">344148</a></p>
</td>
<td class=cellinside>NCKAP5
</td>
<td class=cellinside>NCK associated protein 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4739">4739</a></p>
</td>
<td class=cellinside>NEDD9
</td>
<td class=cellinside>neural precursor cell expressed, developmentally down-regulated 9
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10725">10725</a></p>
</td>
<td class=cellinside>NFAT5
</td>
<td class=cellinside>nuclear factor of activated T cells 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4867">4867</a></p>
</td>
<td class=cellinside>NPHP1
</td>
<td class=cellinside>nephrocystin 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4868">4868</a></p>
</td>
<td class=cellinside>NPHS1
</td>
<td class=cellinside>NPHS1, nephrin
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64111">64111</a></p>
</td>
<td class=cellinside>NPVF
</td>
<td class=cellinside>neuropeptide VF precursor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=11248">11248</a></p>
</td>
<td class=cellinside>NXPH3
</td>
<td class=cellinside>neurexophilin 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54873">54873</a></p>
</td>
<td class=cellinside>PALMD
</td>
<td class=cellinside>palmdelphin
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9749">9749</a></p>
</td>
<td class=cellinside>PHACTR2
</td>
<td class=cellinside>phosphatase and actin regulator 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9050">9050</a></p>
</td>
<td class=cellinside>PSTPIP2
</td>
<td class=cellinside>proline-serine-threonine phosphatase interacting protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9649">9649</a></p>
</td>
<td class=cellinside>RALGPS1
</td>
<td class=cellinside>Ral GEF with PH domain and SH3 binding motif 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6098">6098</a></p>
</td>
<td class=cellinside>ROS1
</td>
<td class=cellinside>ROS proto-oncogene 1, receptor tyrosine kinase
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57190">57190</a></p>
</td>
<td class=cellinside>SEPN1
</td>
<td class=cellinside>selenoprotein N
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10045">10045</a></p>
</td>
<td class=cellinside>SH2D3A
</td>
<td class=cellinside>SH2 domain containing 3A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=400745">400745</a></p>
</td>
<td class=cellinside>SH2D5
</td>
<td class=cellinside>SH2 domain containing 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23616">23616</a></p>
</td>
<td class=cellinside>SH3BP1
</td>
<td class=cellinside>SH3 domain binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=30011">30011</a></p>
</td>
<td class=cellinside>SH3KBP1
</td>
<td class=cellinside>SH3 domain containing kinase binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=112574">112574</a></p>
</td>
<td class=cellinside>SNX18
</td>
<td class=cellinside>sorting nexin 18
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8470">8470</a></p>
</td>
<td class=cellinside>SORBS2
</td>
<td class=cellinside>sorbin and SH3 domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6700">6700</a></p>
</td>
<td class=cellinside>SPRR2A
</td>
<td class=cellinside>small proline rich protein 2A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80725">80725</a></p>
</td>
<td class=cellinside>SRCIN1
</td>
<td class=cellinside>SRC kinase signaling inhibitor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=7039">7039</a></p>
</td>
<td class=cellinside>TGFA
</td>
<td class=cellinside>transforming growth factor alpha
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10188">10188</a></p>
</td>
<td class=cellinside>TNK2
</td>
<td class=cellinside>tyrosine kinase non receptor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84951">84951</a></p>
</td>
<td class=cellinside>TNS4
</td>
<td class=cellinside>tensin 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56995">56995</a></p>
</td>
<td class=cellinside>TULP4
</td>
<td class=cellinside>tubby like protein 4
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>1.45e-01                                                                                                                                                              </td>
<td class=cellinside>6.68e-05                                                                                                                                                              </td>
<td class=cellinside>Reactome                                                                                                                                                              </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-8875513&species=Homo+sapiens&species=Entries+without+species&cluster=true">MET interacts with TNS proteins</a></p></td>
<td class=cellinside>Signal Transduction                                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>3.60e-02                                                                                                                                                              </td>
<td class=cellinside>1.33e-05                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030676">Rac guanyl-nucleotide exchange factor activity</a></p>                                     </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>2.07e-02                                                                                                                                                              </td>
<td class=cellinside>1.86e-04                                                                                                                                                              </td>
<td class=cellinside>Reactome                                                                                                                                                              </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-8875656&species=Homo+sapiens&species=Entries+without+species&cluster=true">MET receptor recycling</a></p>         </td>
<td class=cellinside>Signal Transduction                                                                                                                                                   </td> </tr>
 <tr> 
<td class=cellinside>1.93e-02                                                                                                                                                              </td>
<td class=cellinside>4.04e-11                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0017124">SH3 domain binding</a></p>                                                                 </td>
<td class=cellinside>molecular_function                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>7.11e-03                                                                                                                                                              </td>
<td class=cellinside>1.06e-07                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0007264">small GTPase mediated signal transduction</a></p>                                          </td>
<td class=cellinside>biological_process                                                                                                                                                    </td> </tr>
 <tr> 
<td class=cellinside>6.45e-03                                                                                                                                                              </td>
<td class=cellinside>5.14e-04                                                                                                                                                              </td>
<td class=cellinside>GO                                                                                                                                                                    </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0061099">negative regulation of protein tyrosine kinase activity</a></p>                            </td>
<td class=cellinside>biological_process                                                                                                                                                    </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod362_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>7.28e-10
</td>
<td class=cellinside>3.94e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0017124">SH3 domain binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.84e-07
</td>
<td class=cellinside>9.94e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043087">regulation of GTPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.85e-07
</td>
<td class=cellinside>1.01e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0019904">protein domain specific binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.00e-06
</td>
<td class=cellinside>1.69e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0061097">regulation of protein tyrosine kinase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.72e-06
</td>
<td class=cellinside>1.96e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043547">positive regulation of GTPase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-05
</td>
<td class=cellinside>4.29e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0007588">excretion</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.34e-05
</td>
<td class=cellinside>8.25e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0050730">regulation of peptidyl-tyrosine phosphorylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.19e-05
</td>
<td class=cellinside>1.38e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0061099">negative regulation of protein tyrosine kinase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.78e-05
</td>
<td class=cellinside>1.09e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035591">signaling adaptor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.05e-05
</td>
<td class=cellinside>1.53e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0060090">binding, bridging</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.34e-04
</td>
<td class=cellinside>2.53e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005912">adherens junction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.68e-04
</td>
<td class=cellinside>3.25e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030695">GTPase regulator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.19e-04
</td>
<td class=cellinside>5.66e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0042058">regulation of epidermal growth factor receptor signaling pathway</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.26e-04
</td>
<td class=cellinside>5.81e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0001766">membrane raft polarization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.13e-04
</td>
<td class=cellinside>7.62e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0051665">membrane raft localization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.61e-04
</td>
<td class=cellinside>8.58e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1901184">regulation of ERBB signaling pathway</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.67e-04
</td>
<td class=cellinside>6.42e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008047">enzyme activator activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.71e-04
</td>
<td class=cellinside>8.77e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0050732">negative regulation of peptidyl-tyrosine phosphorylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.16e-04
</td>
<td class=cellinside>7.16e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005085">guanyl-nucleotide exchange factor activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.93e-04
</td>
<td class=cellinside>8.30e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005070">SH3/SH2 adaptor activity</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.00e-03
</td>
<td class=cellinside>1.00e-01
</td>
<td class=cellinside>FACTORS INVOLVED IN MEGAKARYOCYTE DEVELOPMENT AND PLATELET PRODUCTION
</td></tr>
 
<tr> 
<td class=cellinside>1.72e-03
</td>
<td class=cellinside>1.87e-01
</td>
<td class=cellinside>HYALURONAN UPTAKE AND DEGRADATION
</td></tr>
 
<tr> 
<td class=cellinside>3.68e-03
</td>
<td class=cellinside>3.54e-01
</td>
<td class=cellinside>FACTORS INVOLVED IN MEGAKARYOCYTE DEVELOPMENT AND PLATELET PRODUCTION
</td></tr>
 
<tr> 
<td class=cellinside>6.65e-03
</td>
<td class=cellinside>5.77e-01
</td>
<td class=cellinside>HYALURONAN METABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>9.20e-03
</td>
<td class=cellinside>6.85e-01
</td>
<td class=cellinside>SIGNALING BY ILS
</td></tr>
 
<tr> 
<td class=cellinside>1.26e-02
</td>
<td class=cellinside>8.86e-01
</td>
<td class=cellinside>P130CAS LINKAGE TO MAPK SIGNALING FOR INTEGRINS
</td></tr>
 
<tr> 
<td class=cellinside>1.62e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>IL 7 SIGNALING
</td></tr>
 
<tr> 
<td class=cellinside>2.18e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>DEFENSINS
</td></tr>
 
<tr> 
<td class=cellinside>2.21e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNAL ATTENUATION
</td></tr>
 
<tr> 
<td class=cellinside>2.51e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>INTEGRIN ALPHAIIB BETA3 SIGNALING
</td></tr>
 
<tr> 
<td class=cellinside>2.63e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>HYALURONAN UPTAKE AND DEGRADATION
</td></tr>
 
<tr> 
<td class=cellinside>2.86e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>ARMS MEDIATED ACTIVATION
</td></tr>
 
<tr> 
<td class=cellinside>3.04e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>REGULATION OF SIGNALING BY CBL
</td></tr>
 
<tr> 
<td class=cellinside>3.10e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>HYALURONAN METABOLISM
</td></tr>
 
<tr> 
<td class=cellinside>3.21e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PROLONGED ERK ACTIVATION EVENTS
</td></tr>
 
<tr> 
<td class=cellinside>3.33e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>THE ROLE OF NEF IN HIV1 REPLICATION AND DISEASE PATHOGENESIS
</td></tr>
 
<tr> 
<td class=cellinside>3.35e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>NEPHRIN INTERACTIONS
</td></tr>
 
<tr> 
<td class=cellinside>3.69e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNALING BY ILS
</td></tr>
 
<tr> 
<td class=cellinside>3.71e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SIGNAL ATTENUATION
</td></tr>
 
<tr> 
<td class=cellinside>3.93e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PLATELET AGGREGATION PLUG FORMATION
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.88e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010030">abnormal orbit morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.01e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009050">dilated proximal convoluted tubules</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.49e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001402">hypoactivity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.81e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001176">abnormal lung development</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.42e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011427">mesangial cell hyperplasia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.47e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001209">spontaneous skin ulceration</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.58e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000367">abnormal coat/ hair morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009248">small caput epididymis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.75e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011353">expanded mesangial matrix</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.88e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002098">abnormal vibrissa morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.03e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000694">spleen hypoplasia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.19e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009456">impaired cued conditioning behavior</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000019">thick ears</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004285">absent Descemet membrane</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.68e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001458">abnormal object recognition memory</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.11e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005411">delayed fertilization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012165">absent neural folds</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002238">abnormal nasal mucosa morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.88e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008101">lymph node hypoplasia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.11e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009936">abnormal dendritic spine morphology</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 30 17:37:33 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>