<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
<p class='character'><a href="../index.html">Back to main page</a></p>

 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod344</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod344</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>33 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=83638">83638</a></p>
</td>
<td class=cellinside>C11orf68
</td>
<td class=cellinside>chromosome 11 open reading frame 68
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=100506380">100506380</a></p>
</td>
<td class=cellinside>C7orf13
</td>
<td class=cellinside>long intergenic non-protein coding RNA 1006
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10283">10283</a></p>
</td>
<td class=cellinside>CWC27
</td>
<td class=cellinside>CWC27 spliceosome associated protein homolog
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55837">55837</a></p>
</td>
<td class=cellinside>EAPP
</td>
<td class=cellinside>E2F associated phosphoprotein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80153">80153</a></p>
</td>
<td class=cellinside>EDC3
</td>
<td class=cellinside>enhancer of mRNA decapping 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26017">26017</a></p>
</td>
<td class=cellinside>FAM32A
</td>
<td class=cellinside>family with sequence similarity 32 member A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9130">9130</a></p>
</td>
<td class=cellinside>FAM50A
</td>
<td class=cellinside>family with sequence similarity 50 member A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26240">26240</a></p>
</td>
<td class=cellinside>FAM50B
</td>
<td class=cellinside>family with sequence similarity 50 member B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=303741">303741</a></p>
</td>
<td class=cellinside>FRG1
</td>
<td class=cellinside>fascin actin-bundling protein 2, retinal
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=284802">284802</a></p>
</td>
<td class=cellinside>FRG1B
</td>
<td class=cellinside>FSHD region gene 1 family member B, pseudogene
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6936">6936</a></p>
</td>
<td class=cellinside>GCFC2
</td>
<td class=cellinside>GC-rich sequence DNA-binding factor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=22889">22889</a></p>
</td>
<td class=cellinside>KIAA0907
</td>
<td class=cellinside>KH domain containing 4, pre-mRNA splicing factor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=57456">57456</a></p>
</td>
<td class=cellinside>KIAA1143
</td>
<td class=cellinside>KIAA1143
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=4236">4236</a></p>
</td>
<td class=cellinside>MFAP1
</td>
<td class=cellinside>microfibril associated protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80218">80218</a></p>
</td>
<td class=cellinside>NAA50
</td>
<td class=cellinside>N(alpha)-acetyltransferase 50, NatE catalytic subunit
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=222698">222698</a></p>
</td>
<td class=cellinside>NKAPL
</td>
<td class=cellinside>NFKB activating protein like
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55051">55051</a></p>
</td>
<td class=cellinside>NRDE2
</td>
<td class=cellinside>NRDE-2, necessary for RNA interference, domain containing
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51645">51645</a></p>
</td>
<td class=cellinside>PPIL1
</td>
<td class=cellinside>peptidylprolyl isomerase like 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23759">23759</a></p>
</td>
<td class=cellinside>PPIL2
</td>
<td class=cellinside>peptidylprolyl isomerase like 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=53938">53938</a></p>
</td>
<td class=cellinside>PPIL3
</td>
<td class=cellinside>peptidylprolyl isomerase like 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=85313">85313</a></p>
</td>
<td class=cellinside>PPIL4
</td>
<td class=cellinside>peptidylprolyl isomerase like 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79706">79706</a></p>
</td>
<td class=cellinside>PRKRIP1
</td>
<td class=cellinside>PRKR interacting protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=378948">378948</a></p>
</td>
<td class=cellinside>RBMY1B
</td>
<td class=cellinside>RNA binding motif protein, Y-linked, family 1, member B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=378949">378949</a></p>
</td>
<td class=cellinside>RBMY1D
</td>
<td class=cellinside>RNA binding motif protein, Y-linked, family 1, member D
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=378950">378950</a></p>
</td>
<td class=cellinside>RBMY1E
</td>
<td class=cellinside>RNA binding motif protein, Y-linked, family 1, member E
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=378951">378951</a></p>
</td>
<td class=cellinside>RBMY1J
</td>
<td class=cellinside>RNA binding motif protein, Y-linked, family 1, member J
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10169">10169</a></p>
</td>
<td class=cellinside>SERF2
</td>
<td class=cellinside>small EDRK-rich factor 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79753">79753</a></p>
</td>
<td class=cellinside>SNIP1
</td>
<td class=cellinside>Smad nuclear interacting protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51066">51066</a></p>
</td>
<td class=cellinside>SSUH2
</td>
<td class=cellinside>ssu-2 homolog (C. elegans)
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=199745">199745</a></p>
</td>
<td class=cellinside>THAP8
</td>
<td class=cellinside>THAP domain containing 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=59286">59286</a></p>
</td>
<td class=cellinside>UBL5
</td>
<td class=cellinside>ubiquitin like 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79650">79650</a></p>
</td>
<td class=cellinside>USB1
</td>
<td class=cellinside>U6 snRNA biogenesis phosphodiesterase 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23217">23217</a></p>
</td>
<td class=cellinside>ZFR2
</td>
<td class=cellinside>zinc finger RNA binding protein 2
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>3.55e-02                                                                                                                </td>
<td class=cellinside>1.54e-09                                                                                                                </td>
<td class=cellinside>GO                                                                                                                      </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000413">protein peptidyl-prolyl isomerization</a></p></td>
<td class=cellinside>biological_process                                                                                                      </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod344_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>6.39e-11
</td>
<td class=cellinside>5.41e-08
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016071">mRNA metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.15e-10
</td>
<td class=cellinside>9.49e-08
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008380">RNA splicing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.62e-10
</td>
<td class=cellinside>2.84e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006397">mRNA processing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.68e-10
</td>
<td class=cellinside>2.62e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0003755">peptidyl-prolyl cis-trans isomerase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.23e-09
</td>
<td class=cellinside>3.53e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000413">protein peptidyl-prolyl isomerization</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.11e-08
</td>
<td class=cellinside>8.01e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0005681">spliceosomal complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.46e-08
</td>
<td class=cellinside>9.27e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0071013">catalytic step 2 spliceosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.91e-07
</td>
<td class=cellinside>1.02e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016853">isomerase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.78e-06
</td>
<td class=cellinside>7.50e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030529">intracellular ribonucleoprotein complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.82e-06
</td>
<td class=cellinside>7.59e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1990904">ribonucleoprotein complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.52e-06
</td>
<td class=cellinside>1.50e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000398">mRNA splicing, via spliceosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.79e-06
</td>
<td class=cellinside>1.61e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000375">RNA splicing, via transesterification reactions</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.55e-06
</td>
<td class=cellinside>3.01e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006457">protein folding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.42e-04
</td>
<td class=cellinside>3.93e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0035194">posttranscriptional gene silencing by RNA</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.42e-04
</td>
<td class=cellinside>3.93e-02
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016441">posttranscriptional gene silencing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.01e-03
</td>
<td class=cellinside>1.99e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1903955">positive regulation of protein targeting to mitochondrion</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.13e-03
</td>
<td class=cellinside>1.62e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0045171">intercellular bridge</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.36e-03
</td>
<td class=cellinside>2.54e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1903214">regulation of protein targeting to mitochondrion</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.47e-03
</td>
<td class=cellinside>2.70e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031047">gene silencing by RNA</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.15e-03
</td>
<td class=cellinside>3.61e-01
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0010822">positive regulation of mitochondrion organization</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>1.08e-03
</td>
<td class=cellinside>1.26e-01
</td>
<td class=cellinside>BASIGIN INTERACTIONS
</td></tr>
 
<tr> 
<td class=cellinside>8.21e-03
</td>
<td class=cellinside>6.85e-01
</td>
<td class=cellinside>MRNA DECAY BY 5 TO 3 EXORIBONUCLEASE
</td></tr>
 
<tr> 
<td class=cellinside>8.29e-03
</td>
<td class=cellinside>6.30e-01
</td>
<td class=cellinside>BASIGIN INTERACTIONS
</td></tr>
 
<tr> 
<td class=cellinside>1.60e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>MRNA DECAY BY 5 TO 3 EXORIBONUCLEASE
</td></tr>
 
<tr> 
<td class=cellinside>1.68e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>CELL SURFACE INTERACTIONS AT THE VASCULAR WALL
</td></tr>
 
<tr> 
<td class=cellinside>2.10e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>CELL SURFACE INTERACTIONS AT THE VASCULAR WALL
</td></tr>
 
<tr> 
<td class=cellinside>2.29e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>DEADENYLATION DEPENDENT MRNA DECAY
</td></tr>
 
<tr> 
<td class=cellinside>3.48e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>DEADENYLATION DEPENDENT MRNA DECAY
</td></tr>
 
<tr> 
<td class=cellinside>3.73e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>METABOLISM OF MRNA
</td></tr>
 
<tr> 
<td class=cellinside>4.55e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>METABOLISM OF MRNA
</td></tr>
 
<tr> 
<td class=cellinside>4.90e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>METABOLISM OF RNA
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>3.74e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0006241">abnormal placement of pupils</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.40e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010770">preweaning lethality</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.38e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003036">vertebral transformation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.38e-03
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005290">decreased oxygen consumption</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.32e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000150">abnormal rib morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.68e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001231">abnormal epidermis stratum basale morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.68e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005266">abnormal metabolism</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.86e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004647">decreased lumbar vertebrae number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.23e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010101">increased sacral vertebrae number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.62e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003961">decreased lean body mass</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.78e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000480">increased rib number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.99e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004924">abnormal behavior</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.52e-02
</td>
<td class=cellinside>1e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002168">other aberrant phenotype</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 30 17:36:42 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>