<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod102</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod102</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>40 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=53339">53339</a></p>
</td>
<td class=cellinside>BTBD1
</td>
<td class=cellinside>BTB domain containing 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55643">55643</a></p>
</td>
<td class=cellinside>BTBD2
</td>
<td class=cellinside>BTB domain containing 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9318">9318</a></p>
</td>
<td class=cellinside>COPS2
</td>
<td class=cellinside>COP9 signalosome subunit 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8533">8533</a></p>
</td>
<td class=cellinside>COPS3
</td>
<td class=cellinside>COP9 signalosome subunit 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51138">51138</a></p>
</td>
<td class=cellinside>COPS4
</td>
<td class=cellinside>COP9 signalosome subunit 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10987">10987</a></p>
</td>
<td class=cellinside>COPS5
</td>
<td class=cellinside>COP9 signalosome subunit 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10980">10980</a></p>
</td>
<td class=cellinside>COPS6
</td>
<td class=cellinside>COP9 signalosome subunit 6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=50813">50813</a></p>
</td>
<td class=cellinside>COPS7A
</td>
<td class=cellinside>COP9 signalosome subunit 7A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=64708">64708</a></p>
</td>
<td class=cellinside>COPS7B
</td>
<td class=cellinside>COP9 signalosome subunit 7B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10920">10920</a></p>
</td>
<td class=cellinside>COPS8
</td>
<td class=cellinside>COP9 signalosome subunit 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=51185">51185</a></p>
</td>
<td class=cellinside>CRBN
</td>
<td class=cellinside>cereblon
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1447">1447</a></p>
</td>
<td class=cellinside>CSN2
</td>
<td class=cellinside>casein beta
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1448">1448</a></p>
</td>
<td class=cellinside>CSN3
</td>
<td class=cellinside>casein kappa
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8451">8451</a></p>
</td>
<td class=cellinside>CUL4A
</td>
<td class=cellinside>cullin 4A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8450">8450</a></p>
</td>
<td class=cellinside>CUL4B
</td>
<td class=cellinside>cullin 4B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=79269">79269</a></p>
</td>
<td class=cellinside>DCAF10
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 10
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80344">80344</a></p>
</td>
<td class=cellinside>DCAF11
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 11
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=25853">25853</a></p>
</td>
<td class=cellinside>DCAF12
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 12
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=90379">90379</a></p>
</td>
<td class=cellinside>DCAF15
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 15
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54876">54876</a></p>
</td>
<td class=cellinside>DCAF16
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 16
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80067">80067</a></p>
</td>
<td class=cellinside>DCAF17
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 17
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26094">26094</a></p>
</td>
<td class=cellinside>DCAF4
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8816">8816</a></p>
</td>
<td class=cellinside>DCAF5
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55827">55827</a></p>
</td>
<td class=cellinside>DCAF6
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=10238">10238</a></p>
</td>
<td class=cellinside>DCAF7
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=50717">50717</a></p>
</td>
<td class=cellinside>DCAF8
</td>
<td class=cellinside>DDB1 and CUL4 associated factor 8
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=688813">688813</a></p>
</td>
<td class=cellinside>DDA1
</td>
<td class=cellinside>DET1 and DDB1 associated 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1642">1642</a></p>
</td>
<td class=cellinside>DDB1
</td>
<td class=cellinside>damage specific DNA binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=1643">1643</a></p>
</td>
<td class=cellinside>DDB2
</td>
<td class=cellinside>damage specific DNA binding protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6468">6468</a></p>
</td>
<td class=cellinside>FBXW4
</td>
<td class=cellinside>F-box and WD repeat domain containing 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=2873">2873</a></p>
</td>
<td class=cellinside>GPS1
</td>
<td class=cellinside>G protein pathway suppressor 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=130535">130535</a></p>
</td>
<td class=cellinside>KCTD18
</td>
<td class=cellinside>potassium channel tetramerization domain containing 18
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=80311">80311</a></p>
</td>
<td class=cellinside>KLHL15
</td>
<td class=cellinside>kelch like family member 15
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54800">54800</a></p>
</td>
<td class=cellinside>KLHL24
</td>
<td class=cellinside>kelch like family member 24
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9684">9684</a></p>
</td>
<td class=cellinside>LRRC14
</td>
<td class=cellinside>leucine rich repeat containing 14
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55023">55023</a></p>
</td>
<td class=cellinside>PHIP
</td>
<td class=cellinside>pleckstrin homology domain interacting protein
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9840">9840</a></p>
</td>
<td class=cellinside>TESPA1
</td>
<td class=cellinside>thymocyte expressed, positive selection associated 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=387357">387357</a></p>
</td>
<td class=cellinside>THEMIS
</td>
<td class=cellinside>thymocyte selection associated
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=163590">163590</a></p>
</td>
<td class=cellinside>TOR1AIP2
</td>
<td class=cellinside>torsin 1A interacting protein 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=26133">26133</a></p>
</td>
<td class=cellinside>TRPC4AP
</td>
<td class=cellinside>transient receptor potential cation channel subfamily C member 4 associated protein
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>5.33e-01                                                                                                                                                               </td>
<td class=cellinside>5.91e-24                                                                                                                                                               </td>
<td class=cellinside>GO                                                                                                                                                                     </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000338">protein deneddylation</a></p>                                                               </td>
<td class=cellinside>biological_process                                                                                                                                                     </td> </tr>
 <tr> 
<td class=cellinside>4.09e-01                                                                                                                                                               </td>
<td class=cellinside>4.15e-37                                                                                                                                                               </td>
<td class=cellinside>GO                                                                                                                                                                     </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0080008">Cul4-RING E3 ubiquitin ligase complex</a></p>                                               </td>
<td class=cellinside>cellular_component                                                                                                                                                     </td> </tr>
 <tr> 
<td class=cellinside>8.76e-02                                                                                                                                                               </td>
<td class=cellinside>5.06e-30                                                                                                                                                               </td>
<td class=cellinside>GO                                                                                                                                                                     </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000715">nucleotide-excision repair, DNA damage recognition</a></p>                                  </td>
<td class=cellinside>biological_process                                                                                                                                                     </td> </tr>
 <tr> 
<td class=cellinside>4.92e-02                                                                                                                                                               </td>
<td class=cellinside>1.10e-29                                                                                                                                                               </td>
<td class=cellinside>Reactome                                                                                                                                                               </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-5696394&species=Homo+sapiens&species=Entries+without+species&cluster=true">DNA Damage Recognition in GG-NER</a></p></td>
<td class=cellinside>DNA Repair                                                                                                                                                             </td> </tr>
 <tr> 
<td class=cellinside>3.29e-02                                                                                                                                                               </td>
<td class=cellinside>1.44e-21                                                                                                                                                               </td>
<td class=cellinside>GO                                                                                                                                                                     </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008180">COP9 signalosome</a></p>                                                                    </td>
<td class=cellinside>cellular_component                                                                                                                                                     </td> </tr>
 <tr> 
<td class=cellinside>4.39e-03                                                                                                                                                               </td>
<td class=cellinside>7.84e-38                                                                                                                                                               </td>
<td class=cellinside>Reactome                                                                                                                                                               </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-8951664&species=Homo+sapiens&species=Entries+without+species&cluster=true">Neddylation</a></p>                     </td>
<td class=cellinside>Metabolism of proteins                                                                                                                                                 </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod102_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016567">protein ubiquitination</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032446">protein modification by small protein conjugation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006281">DNA repair</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006283">transcription-coupled nucleotide-excision repair</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006289">nucleotide-excision repair</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000715">nucleotide-excision repair, DNA damage recognition</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000338">protein deneddylation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A1990234">transferase complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000151">ubiquitin ligase complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031461">cullin-RING ubiquitin ligase complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0080008">Cul4-RING E3 ubiquitin ligase complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008180">COP9 signalosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.11e-10
</td>
<td class=cellinside>3.93e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006974">cellular response to DNA damage stimulus</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.51e-09
</td>
<td class=cellinside>1.10e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0070646">protein modification by small protein removal</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.43e-09
</td>
<td class=cellinside>3.02e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006259">DNA metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.09e-08
</td>
<td class=cellinside>7.06e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0000717">nucleotide-excision repair, DNA duplex unwinding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.91e-08
</td>
<td class=cellinside>1.78e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006511">ubiquitin-dependent protein catabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.33e-08
</td>
<td class=cellinside>2.02e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0019941">modification-dependent protein catabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.95e-08
</td>
<td class=cellinside>2.13e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0031464">Cul4A-RING E3 ubiquitin ligase complex</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.51e-08
</td>
<td class=cellinside>3.79e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006294">nucleotide-excision repair, preincision complex assembly</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>3.00e-05
</td>
<td class=cellinside>5.27e-03
</td>
<td class=cellinside>FORMATION OF INCISION COMPLEX IN GG NER
</td></tr>
 
<tr> 
<td class=cellinside>3.07e-05
</td>
<td class=cellinside>5.38e-03
</td>
<td class=cellinside>GLOBAL GENOMIC NER GG NER
</td></tr>
 
<tr> 
<td class=cellinside>4.48e-05
</td>
<td class=cellinside>7.58e-03
</td>
<td class=cellinside>NUCLEOTIDE EXCISION REPAIR
</td></tr>
 
<tr> 
<td class=cellinside>9.22e-05
</td>
<td class=cellinside>1.18e-02
</td>
<td class=cellinside>FORMATION OF INCISION COMPLEX IN GG NER
</td></tr>
 
<tr> 
<td class=cellinside>9.45e-05
</td>
<td class=cellinside>1.21e-02
</td>
<td class=cellinside>GLOBAL GENOMIC NER GG NER
</td></tr>
 
<tr> 
<td class=cellinside>1.87e-04
</td>
<td class=cellinside>2.23e-02
</td>
<td class=cellinside>NUCLEOTIDE EXCISION REPAIR
</td></tr>
 
<tr> 
<td class=cellinside>8.74e-04
</td>
<td class=cellinside>8.92e-02
</td>
<td class=cellinside>DNA REPAIR
</td></tr>
 
<tr> 
<td class=cellinside>1.47e-03
</td>
<td class=cellinside>1.64e-01
</td>
<td class=cellinside>DNA REPAIR
</td></tr>
 
<tr> 
<td class=cellinside>5.58e-03
</td>
<td class=cellinside>4.48e-01
</td>
<td class=cellinside>NUCLEAR SIGNALING BY ERBB4
</td></tr>
 
<tr> 
<td class=cellinside>8.89e-03
</td>
<td class=cellinside>6.68e-01
</td>
<td class=cellinside>SIGNALING BY ERBB4
</td></tr>
 
<tr> 
<td class=cellinside>9.88e-03
</td>
<td class=cellinside>7.93e-01
</td>
<td class=cellinside>NUCLEAR SIGNALING BY ERBB4
</td></tr>
 
<tr> 
<td class=cellinside>9.94e-03
</td>
<td class=cellinside>7.97e-01
</td>
<td class=cellinside>SIGNALING BY ERBB4
</td></tr>
 
<tr> 
<td class=cellinside>3.74e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>ASSOCIATION OF TRIC CCT WITH TARGET PROTEINS DURING BIOSYNTHESIS
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>7.05e-05
</td>
<td class=cellinside>1.79e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0013588">small thymus medulla</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.13e-04
</td>
<td class=cellinside>6.83e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004047">abnormal milk composition</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.57e-04
</td>
<td class=cellinside>7.65e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008049">increased memory T cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.06e-03
</td>
<td class=cellinside>9.29e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004974">decreased regulatory T cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.95e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008058">abnormal DNA repair</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.40e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001696">failure to gastrulate</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.57e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002145">abnormal T cell differentiation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011112">partial lethality during fetal growth through weaning</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010382">abnormal dosage compensation, by inactivation of X chromosome</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004941">abnormal regulatory T cell morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.62e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003671">abnormal eyelid aperture</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.58e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008079">decreased CD8-positive, alpha-beta T cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.02e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008075">decreased CD4-positive, alpha beta T cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.72e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0013504">increased embryonic tissue cell apoptosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.33e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0010039">abnormal trophoblast giant cell proliferation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004228">decreased cellular sensitivity to ionizing radiation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004501">increased incidence of tumors by UV-induction</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008858">abnormal hair cycle anagen phase</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011527">disorganized placental labyrinth</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004266">pale placenta</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 23 09:49:26 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>