<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
<head>
<title> R output </title>
<link rel=stylesheet href="R2HTML_tao.css" type=text/css>
<object id="mathplayer" classid="clsid:32F66A20-7614-11D4-BD11-00104BD3F987"></object>
<?import namespace="mml" implementation="#mathplayer"?>
<script type="text/javascript" src="ASCIIMathML.js"></script>
<link href="./runtime/styles/xp/grid.css" rel="stylesheet" type="text/css" ></link>
<link href="gridR2HTML.css" rel="stylesheet" type="text/css" ></link>

<script src="./runtime/lib/grid.js"></script>

<script src="./gridR2HTML.js"></script>
<script>
   nequations=0;
</script>
</head>
<body onload="translate()" bgcolor= FFFFFF >
 <h1 > DREAM Module Identification Challenge – Consensus modules</h1>

 <h2 > PPI-STRING_Consensus_mod215</h2>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr> 
<tr><td class=firstcolumn>Assigned name</td>
<td class=cellinside>NA                         </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Network</td>
<td class=cellinside>PPI-STRING                 </td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module ID</td>
<td class=cellinside>PPI-STRING_Consensus_mod215</td> </tr>
 <tr> 
<tr><td class=firstcolumn>Module size</td>
<td class=cellinside>21 genes                   </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Module genes</h2>

 <h3 > This module comprises the following genes:</h3>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>Gene ID  </th>
		<th>Gene Symbol  </th>
		<th>Gene Name</th> 
	</tr> 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=165">165</a></p>
</td>
<td class=cellinside>AEBP1
</td>
<td class=cellinside>AE binding protein 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=56265">56265</a></p>
</td>
<td class=cellinside>CPXM1
</td>
<td class=cellinside>carboxypeptidase X, M14 family member 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=119587">119587</a></p>
</td>
<td class=cellinside>CPXM2
</td>
<td class=cellinside>carboxypeptidase X, M14 family member 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=8532">8532</a></p>
</td>
<td class=cellinside>CPZ
</td>
<td class=cellinside>carboxypeptidase Z
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5045">5045</a></p>
</td>
<td class=cellinside>FURIN
</td>
<td class=cellinside>furin, paired basic amino acid cleaving enzyme
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5122">5122</a></p>
</td>
<td class=cellinside>PCSK1
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=27344">27344</a></p>
</td>
<td class=cellinside>PCSK1N
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 1 inhibitor
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5126">5126</a></p>
</td>
<td class=cellinside>PCSK2
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 2
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54760">54760</a></p>
</td>
<td class=cellinside>PCSK4
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5125">5125</a></p>
</td>
<td class=cellinside>PCSK5
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 5
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5046">5046</a></p>
</td>
<td class=cellinside>PCSK6
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 6
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=9159">9159</a></p>
</td>
<td class=cellinside>PCSK7
</td>
<td class=cellinside>proprotein convertase subtilisin/kexin type 7
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=5545">5545</a></p>
</td>
<td class=cellinside>PRB4
</td>
<td class=cellinside>proline rich protein BstNI subfamily 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=284654">284654</a></p>
</td>
<td class=cellinside>RSPO1
</td>
<td class=cellinside>R-spondin 1
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=84870">84870</a></p>
</td>
<td class=cellinside>RSPO3
</td>
<td class=cellinside>R-spondin 3
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=343637">343637</a></p>
</td>
<td class=cellinside>RSPO4
</td>
<td class=cellinside>R-spondin 4
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=6447">6447</a></p>
</td>
<td class=cellinside>SCG5
</td>
<td class=cellinside>secretogranin V
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=23230">23230</a></p>
</td>
<td class=cellinside>VPS13A
</td>
<td class=cellinside>vacuolar protein sorting 13 homolog A
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=157680">157680</a></p>
</td>
<td class=cellinside>VPS13B
</td>
<td class=cellinside>vacuolar protein sorting 13 homolog B
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=54832">54832</a></p>
</td>
<td class=cellinside>VPS13C
</td>
<td class=cellinside>vacuolar protein sorting 13 homolog C
</td></tr>
 
<tr> 
<td class=cellinside><a href="https://www.ncbi.nlm.nih.gov/gene/?term=55187">55187</a></p>
</td>
<td class=cellinside>VPS13D
</td>
<td class=cellinside>vacuolar protein sorting 13 homolog D
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

 <h2 > Functional annotation</h2>

<p class='character'><p>Modules were tested for enrichment in functional and pathway annotations using two complementary approaches:
         <br>    1. To select a small number of specific / non-redundant annotations for each module, a regression-based approach was used;
         <br>    2. To obtain the complete set of enriched annotations, an extension of Fisher’s exact test that takes annotation bias into account was employed (Wallenius’ non-central hypergeometric distribution).
         </p></p>

 <h3 > Most specific annotations for this module</h3>


<p align= center >
<table cellspacing=0 border=1><caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> <tr class= firstline > <th>Coeff.<sup>1</sup></th><th>P-value.<sup>2</sup></th><th>Source.<sup>3</sup></th><th>Term.<sup>4</sup></th><th>Class.<sup>5</sup></th> </tr>
 <tr> 
<td class=cellinside>2.38e-01                                                                                                                                                                         </td>
<td class=cellinside>1.75e-20                                                                                                                                                                         </td>
<td class=cellinside>GO                                                                                                                                                                               </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016486">peptide hormone processing</a></p>                                                                    </td>
<td class=cellinside>biological_process                                                                                                                                                               </td> </tr>
 <tr> 
<td class=cellinside>1.31e-01                                                                                                                                                                         </td>
<td class=cellinside>4.59e-09                                                                                                                                                                         </td>
<td class=cellinside>Reactome                                                                                                                                                                         </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-9036866&species=Homo+sapiens&species=Entries+without+species&cluster=true">Expression and Processing of Neurotrophins</a></p></td>
<td class=cellinside>Signal Transduction                                                                                                                                                              </td> </tr>
 <tr> 
<td class=cellinside>1.30e-01                                                                                                                                                                         </td>
<td class=cellinside>4.59e-09                                                                                                                                                                         </td>
<td class=cellinside>Reactome                                                                                                                                                                         </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-167060&species=Homo+sapiens&species=Entries+without+species&cluster=true">NGF processing</a></p>                             </td>
<td class=cellinside>Signal Transduction                                                                                                                                                              </td> </tr>
 <tr> 
<td class=cellinside>1.08e-01                                                                                                                                                                         </td>
<td class=cellinside>1.06e-09                                                                                                                                                                         </td>
<td class=cellinside>GO                                                                                                                                                                               </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006518">peptide metabolic process</a></p>                                                                     </td>
<td class=cellinside>biological_process                                                                                                                                                               </td> </tr>
 <tr> 
<td class=cellinside>4.38e-02                                                                                                                                                                         </td>
<td class=cellinside>6.33e-19                                                                                                                                                                         </td>
<td class=cellinside>GO                                                                                                                                                                               </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016485">protein processing</a></p>                                                                            </td>
<td class=cellinside>biological_process                                                                                                                                                               </td> </tr>
 <tr> 
<td class=cellinside>2.55e-02                                                                                                                                                                         </td>
<td class=cellinside>1.51e-06                                                                                                                                                                         </td>
<td class=cellinside>Reactome                                                                                                                                                                         </td>
<td class=cellinside><a href="https://reactome.org/content/query?q=R-HSA-4641263&species=Homo+sapiens&species=Entries+without+species&cluster=true">Regulation of FZD by ubiquitination</a></p>       </td>
<td class=cellinside>Signal Transduction                                                                                                                                                              </td> </tr>
 <tr> 
<td class=cellinside>5.18e-04                                                                                                                                                                         </td>
<td class=cellinside>2.15e-08                                                                                                                                                                         </td>
<td class=cellinside>GO                                                                                                                                                                               </td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004181">metallocarboxypeptidase activity</a></p>                                                              </td>
<td class=cellinside>molecular_function                                                                                                                                                               </td> </tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>Regression coefficient</p></p>

<p class='character'><p><sup>2</sup>Fisher’s exact test nominal P-value</p></p>

<p class='character'><p><sup>3</sup>Annotation source (Reactome, GO biological process (BP), molecular function (MF) and cellular component (CC))</p></p>

<p class='character'><p><sup>4</sup>GO category or Reactome pathway</p></p>

<p class='character'><p><sup>5</sup>High-level branch of annotation tree</p></p>

 <h3 > Gene membership</h3>

<p align=center><img src='PPI-STRING_Consensus_mod215_pathway2gene.PNG' border=1 width=1200>
 <h3 > All enriched annotations</h3>

 <h4 > Gene Ontology</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0043603">cellular amide metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0006518">peptide metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016485">protein processing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0051604">protein maturation</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0016486">peptide hormone processing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>0.00e+00
</td>
<td class=cellinside>0.00e+00
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0042445">hormone metabolic process</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-10
</td>
<td class=cellinside>3.10e-07
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008233">peptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.12e-09
</td>
<td class=cellinside>2.18e-06
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0010817">regulation of hormone levels</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.48e-08
</td>
<td class=cellinside>1.44e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004252">serine-type endopeptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.84e-08
</td>
<td class=cellinside>2.87e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0032455">nerve growth factor processing</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.89e-08
</td>
<td class=cellinside>2.34e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008236">serine-type peptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.79e-08
</td>
<td class=cellinside>2.67e-05
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0017171">serine hydrolase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.77e-07
</td>
<td class=cellinside>2.82e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004181">metallocarboxypeptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.01e-06
</td>
<td class=cellinside>2.95e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030141">secretory granule</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.26e-06
</td>
<td class=cellinside>6.72e-04
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004180">carboxypeptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>3.65e-06
</td>
<td class=cellinside>1.04e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0004175">endopeptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>6.82e-06
</td>
<td class=cellinside>1.84e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008235">metalloexopeptidase activity</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.42e-06
</td>
<td class=cellinside>3.67e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0030177">positive regulation of Wnt signaling pathway</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.62e-05
</td>
<td class=cellinside>6.31e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0048406">nerve growth factor binding</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.94e-05
</td>
<td class=cellinside>7.02e-03
</td>
<td class=cellinside><a href="http://amigo.geneontology.org/amigo/medial_search?q=GO%3A0008201">heparin binding</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
             </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
             </p>

 <h4 > Reactome</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>7.97e-05
</td>
<td class=cellinside>1.27e-02
</td>
<td class=cellinside>SIGNALLING BY NGF
</td></tr>
 
<tr> 
<td class=cellinside>9.23e-05
</td>
<td class=cellinside>1.44e-02
</td>
<td class=cellinside>SIGNALING BY NODAL
</td></tr>
 
<tr> 
<td class=cellinside>2.69e-04
</td>
<td class=cellinside>3.72e-02
</td>
<td class=cellinside>INSULIN SYNTHESIS AND PROCESSING
</td></tr>
 
<tr> 
<td class=cellinside>3.31e-04
</td>
<td class=cellinside>3.75e-02
</td>
<td class=cellinside>SIGNALING BY NODAL
</td></tr>
 
<tr> 
<td class=cellinside>3.66e-04
</td>
<td class=cellinside>4.10e-02
</td>
<td class=cellinside>INSULIN SYNTHESIS AND PROCESSING
</td></tr>
 
<tr> 
<td class=cellinside>2.07e-03
</td>
<td class=cellinside>1.92e-01
</td>
<td class=cellinside>SIGNALLING BY NGF
</td></tr>
 
<tr> 
<td class=cellinside>5.42e-03
</td>
<td class=cellinside>4.39e-01
</td>
<td class=cellinside>GAMMA CARBOXYLATION TRANSPORT AND AMINO TERMINAL CLEAVAGE OF PROTEINS
</td></tr>
 
<tr> 
<td class=cellinside>6.96e-03
</td>
<td class=cellinside>5.45e-01
</td>
<td class=cellinside>DIABETES PATHWAYS
</td></tr>
 
<tr> 
<td class=cellinside>7.22e-03
</td>
<td class=cellinside>5.62e-01
</td>
<td class=cellinside>PEPTIDE HORMONE BIOSYNTHESIS
</td></tr>
 
<tr> 
<td class=cellinside>7.76e-03
</td>
<td class=cellinside>6.55e-01
</td>
<td class=cellinside>DIABETES PATHWAYS
</td></tr>
 
<tr> 
<td class=cellinside>1.30e-02
</td>
<td class=cellinside>9.80e-01
</td>
<td class=cellinside>PEPTIDE HORMONE BIOSYNTHESIS
</td></tr>
 
<tr> 
<td class=cellinside>1.34e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>GAMMA CARBOXYLATION TRANSPORT AND AMINO TERMINAL CLEAVAGE OF PROTEINS
</td></tr>
 
<tr> 
<td class=cellinside>1.46e-02
</td>
<td class=cellinside>9.94e-01
</td>
<td class=cellinside>PTM GAMMA CARBOXYLATION HYPUSINE FORMATION AND ARYLSULFATASE ACTIVATION
</td></tr>
 
<tr> 
<td class=cellinside>1.49e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SYNTHESIS SECRETION AND INACTIVATION OF GIP
</td></tr>
 
<tr> 
<td class=cellinside>1.53e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>TGF BETA RECEPTOR SIGNALING ACTIVATES SMADS
</td></tr>
 
<tr> 
<td class=cellinside>1.62e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SYNTHESIS SECRETION AND INACTIVATION OF GIP
</td></tr>
 
<tr> 
<td class=cellinside>1.80e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PRE NOTCH PROCESSING IN GOLGI
</td></tr>
 
<tr> 
<td class=cellinside>1.83e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>SYNTHESIS SECRETION AND INACTIVATION OF GLP1
</td></tr>
 
<tr> 
<td class=cellinside>1.94e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>INCRETIN SYNTHESIS SECRETION AND INACTIVATION
</td></tr>
 
<tr> 
<td class=cellinside>2.22e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside>PRE NOTCH PROCESSING IN GOLGI
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
</p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
</p>

 <h4 > Mouse mutant phenotypes</h4>


<p align= center >
<table cellspacing=0 border=1>
<caption align=bottom class=captiondataframe></caption>
<tr><td>
	<table border=0 class=dataframe>
	<tbody> 
	<tr class= firstline > 
		<th>P-value<sup>1<sup>  </th>
		<th>FDR<sup>2<sup>  </th>
		<th>Term</th> 
	</tr> 
<tr> 
<td class=cellinside>6.69e-04
</td>
<td class=cellinside>7.19e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004179">transmission ratio distortion</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.00e-03
</td>
<td class=cellinside>9.02e-01
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002696">decreased circulating glucagon level</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>2.42e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0000644">dextrocardia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.37e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0002633">persistent truncus arteriosis</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>4.85e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0005215">abnormal pancreatic islet morphology</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009706">absent midgut</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>5.61e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009282">reduced hyperactivated sperm motility</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0009180">increased pancreatic delta cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>7.48e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0012084">truncated foregut</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>8.87e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001433">polyphagia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.34e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008200">decreased follicular dendritic cell number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>9.36e-03
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001665">chronic diarrhea</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004270">analgesia</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004656">absent sacral vertebrae</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004694">absent patella</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0008999">absent anus</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.12e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0003582">abnormal ovary development</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.21e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0001923">reduced female fertility</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.30e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0011121">decreased primordial ovarian follicle number</a></p>
</td></tr>
 
<tr> 
<td class=cellinside>1.31e-02
</td>
<td class=cellinside>1.00e+00
</td>
<td class=cellinside><a href="http://www.informatics.jax.org/vocab/mp_ontology/MP:0004653">absent caudal vertebrae</a></p>
</td></tr>
 
	</tbody>
</table>
 </td></table>
 <br>

<p class='character'><p><sup>1</sup>1Nominal enrichment p-value (Wallenius’ noncentral hypergeometric distribution)
                 </p></p>

<p class='character'><p><sup>2</sup>FDR corrected p-value (Benjamini-Hochberg)
                 </p>

<hr size=1>
<font size=-1>
	 Generated on: <i>Thu Aug 23 09:55:04 2018</i> - <b>R2HTML</b> 
<hr size=1>
	</body>
</html>